From 086ec61f141e0882aa7b833446ec44774228bd0c Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 10 Apr 2018 12:23:10 +0000
Subject: [PATCH] Update CLI version in global.json

---
 global.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/global.json b/global.json
index ec01c56..72338ce 100644
--- a/global.json
+++ b/global.json
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.300-preview2-008324"
+    "version": "2.1.300-preview2-008533"
   }
 }
-- 
1.8.3.1

