From 4aafe8d8f3c4901af7fd668ebcc268fec8277201 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 18 Apr 2019 14:27:10 -0500
Subject: [PATCH] Add repository properties for new SDK warning.

---
 Directory.Build.props | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index d7042790ef..fca991ca68 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -2,5 +2,10 @@
 <Project>
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   <Import Project="eng\targets\Settings.props" />
+  <PropertyGroup>
+    <PackageProjectUrl>https://github.com/dotnet/roslyn</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/dotnet/roslyn</RepositoryUrl>
+    <RepositoryType>Git</RepositoryType>
+  </PropertyGroup>
 </Project>
 
-- 
2.18.0

