From e9e3cdf532000c5e8d10e5df65f94e06ab8657e4 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 15 Jul 2019 19:26:42 +0000
Subject: [PATCH] Conditionally remove net472 from TargetFrameworks

---
 .../Microsoft.Net.Compilers.Toolset.Package.csproj                       | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj b/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
index 1ae79d8..486b1af 100644
--- a/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
+++ b/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
@@ -2,6 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1;net472</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">netcoreapp2.1</TargetFrameworks>
     <RoslynProjectType>Custom</RoslynProjectType>
 
     <IsPackable>true</IsPackable>
-- 
1.8.3.1

