From e18cff2b8d9ca8bca3c8c36c55e47a3d81a9eb20 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 2 Oct 2017 11:15:41 -0500
Subject: [PATCH] Only import BuildTools.props if it exists.

---
 build/Targets/Settings.props | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build/Targets/Settings.props b/build/Targets/Settings.props
index 3592746db7..8ce599195e 100644
--- a/build/Targets/Settings.props
+++ b/build/Targets/Settings.props
@@ -141,7 +141,8 @@
   <Import Project="$(RoslynDiagnosticsPropsFilePath)" Condition="'$(UseRoslynAnalyzers)' == 'true' AND Exists('$(RoslynDiagnosticsPropsFilePath)')" />
 
   <ImportGroup Label="WindowsImports" Condition="'$(OS)' == 'Windows_NT'">
-    <Import Project="$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\$(VisualStudioBuildToolsVersion)\build\Microsoft.VsSDK.BuildTools.props" />
+    <Import Project="$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\$(VisualStudioBuildToolsVersion)\build\Microsoft.VsSDK.BuildTools.props" 
+            Condition="Exists('$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\$(VisualStudioBuildToolsVersion)\build\Microsoft.VsSDK.BuildTools.props')" />
   </ImportGroup>
 
   <Import Project="$(ToolsetCompilerPropsFilePath)" Condition="Exists('$(ToolsetCompilerPropsFilePath)') AND '$(BootstrapBuildPath)' == ''" />
-- 
2.12.2.windows.2

