From 4bf4f5c1812eda9ff009449b524e9f3069574ac3 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Thu, 19 Sep 2019 15:39:49 +0000
Subject: [PATCH] Import PackageVersions props if exists

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index aed48a0..412101a 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -343,4 +343,5 @@
       https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
 </Project>
-- 
1.8.3.1

