From 47c14f1f0f0523aa4e86b24424013073b4048eb9 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 22 May 2017 14:07:59 -0700
Subject: [PATCH 4/5] Copy prebuilt dotnet from known location

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7cad416..bbab296 100644
--- a/Makefile
+++ b/Makefile
@@ -64,11 +64,7 @@ restore: $(DOTNET)
 $(DOTNET):
 	mkdir -p $(BINARIES_PATH) ; \
 	pushd $(BINARIES_PATH) ; \
-	curl -O https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh && \
-	chmod +x dotnet-install.sh && \
-	./dotnet-install.sh --version "$(DOTNET_VERSION)" --install-dir "$(BINARIES_PATH)/dotnet-cli"
-
+	cp -r $(DOTNET_TOOL_DIR) $(BINARIES_PATH)/dotnet-cli
 
 clean:
 	@rm -rf Binaries
-- 
1.8.3.1

