From db0b16b469094e8f37623cdf86902491f502d575 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Wed, 20 Nov 2019 22:32:33 +0000
Subject: [PATCH 5/6] Import PackageVersions props if exists

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 87c3523..eecfde6 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -322,4 +322,5 @@
     -->
     <UsingToolMicrosoftNetCompilers Condition="'$(BootstrapBuildPath)' == ''">true</UsingToolMicrosoftNetCompilers>
   </PropertyGroup>
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
 </Project>
-- 
1.8.3.1

