From 93a4b6a0a67a211a2e69b198b0e6a98a941d64ce Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 4 Sep 2020 09:32:49 -0500
Subject: [PATCH 5/5] Don't restore tools in source-build

---
 eng/build.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eng/build.sh b/eng/build.sh
index 30a6fbd1f8e..bf361f0a497 100755
--- a/eng/build.sh
+++ b/eng/build.sh
@@ -298,9 +298,9 @@ function BuildSolution {
 }
 
 InitializeDotNetCli $restore
-if [[ "$restore" == true ]]; then
-  dotnet tool restore
-fi
+#if [[ "$restore" == true ]]; then
+#  dotnet tool restore
+#fi
 
 bootstrap_dir=""
 if [[ "$bootstrap" == true ]]; then
-- 
2.18.0

