From 4bc36ffcdd6ea3434c39a7e7aede43d16f38c6a8 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 4 Sep 2019 15:49:31 +0000
Subject: [PATCH] Import PackageVersions props if exists

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 318c913..d63c38e 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -333,4 +333,5 @@
       https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json;
     </RestoreSources>
   </PropertyGroup>
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
 </Project>
-- 
1.8.3.1

