diff --git a/Makefile b/Makefile
index d139d29..05f769d 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,7 @@ NUGET_EXE = $(shell pwd)/nuget.exe
 MSBUILD_ADDITIONALARGS := /v:m /fl /fileloggerparameters:Verbosity=normal /p:Configuration=$(BUILD_CONFIGURATION)
 
 ifeq ($(OS_NAME),Linux)
-	MSBUILD_ADDITIONALARGS := $(MSBUILD_ADDITIONALARGS) /p:BaseNuGetRuntimeIdentifier=$(shell . /etc/os-release && echo $$ID.$$VERSION_ID)
+	MSBUILD_ADDITIONALARGS := $(MSBUILD_ADDITIONALARGS) /p:BaseNuGetRuntimeIdentifier=$(shell ([ -z "$$DOTNET_RUNTIME_ID" ] && ( . /etc/os-release ; echo "$$ID.$$VERSION_ID")) || (echo "$$DOTNET_RUNTIME_ID" | sed 's/-.*//g'))
 else ifeq ($(OS_NAME),Darwin)
 	MSBUILD_ADDITIONALARGS := $(MSBUILD_ADDITIONALARGS) /p:BaseNuGetRuntimeIdentifier=osx.10.12
 endif
