From 61b75d0a0ffdee7ebee0d23a096d84c8f67dff48 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 12 Sep 2019 19:47:14 -0500
Subject: [PATCH 2/2] Try extra props

---
 .../src/Microsoft.Extensions.FileProviders.Embedded.csproj     | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj b/src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj
index d36323e3b..c0182794e 100644
--- a/src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj
+++ b/src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj
@@ -28,8 +28,11 @@
   <ItemGroup>
     <NuspecProperty Include="AssemblyName=$(AssemblyName)" />
     <NuspecProperty Include="OutputBinary=$(OutputPath)**\$(AssemblyName).dll" />
+    <NuspecProperty Include="OutputBinary=$(OutputPath)**/*.dll" />
     <NuspecProperty Include="OutputSymbol=$(OutputPath)**\$(AssemblyName).pdb" />
+    <NuspecProperty Include="OutputSymbol=$(OutputPath)**/*.pdb" />
     <NuspecProperty Include="OutputDocumentation=$(OutputPath)**\$(AssemblyName).xml" />
+    <NuspecProperty Include="OutputDocumentation=$(OutputPath)**/*.xml" />
     <NuspecProperty Include="TaskAssemblyNetStandard=$(ArtifactsDir)bin\$(AssemblyName).Manifest.Task\$(Configuration)\netstandard2.0\$(AssemblyName).Manifest.Task.dll"/>
     <NuspecProperty Include="TaskSymbolNetStandard=$(ArtifactsDir)bin\$(AssemblyName).Manifest.Task\$(Configuration)\netstandard2.0\$(AssemblyName).Manifest.Task.pdb" Condition="'$(DebugType)'!='embedded'"/>
   </ItemGroup>
-- 
2.17.1

