From d6e9705b28593aec6873415b79ea9484bab202cd Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 8 Nov 2019 14:35:59 -0600
Subject: [PATCH 3/4] Make ref assembly versions match implementation assembly
 versions

---
 eng/Versions.props | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 4e5a0f459..87f6bb05a 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -11,13 +11,7 @@
     <PatchVersion>2</PatchVersion>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
     <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
-    <AssemblyVersion Condition="'$(IsReferenceAssemblyProject)' != 'true'">$(VersionPrefix).0</AssemblyVersion>
-    <!--
-      We do not support changing reference assemblies in a patch. This ignores
-      the patch version number to ensure assembly version of ref assemblies stays constant
-      throughout major.minor.
-    -->
-    <AssemblyVersion Condition="'$(IsReferenceAssemblyProject)' == 'true'">$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
+    <AssemblyVersion>$(VersionPrefix).0</AssemblyVersion>
     <ExperimentalVersionPrefix>0.1.$(PatchVersion)</ExperimentalVersionPrefix>
     <!--
       Until package baselines are updated (see aspnet/AspNetCore#12702), ignore them. This also
-- 
2.18.0

