From 35d5223d79da8276d2bc56be7832590dce878269 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 7 Nov 2019 02:48:29 -0600
Subject: [PATCH 3/3] Remove duplicate source file

---
 .../src/Microsoft.Extensions.Logging.Configuration.csproj       | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/Logging/Logging.Configuration/src/Microsoft.Extensions.Logging.Configuration.csproj b/src/Logging/Logging.Configuration/src/Microsoft.Extensions.Logging.Configuration.csproj
index 641424cfc..b46b3119a 100644
--- a/src/Logging/Logging.Configuration/src/Microsoft.Extensions.Logging.Configuration.csproj
+++ b/src/Logging/Logging.Configuration/src/Microsoft.Extensions.Logging.Configuration.csproj
@@ -9,8 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="$(SharedSourceRoot)ProviderAliasUtilities\*.cs" />
-
     <Reference Include="Microsoft.Extensions.Logging" />
     <Reference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
   </ItemGroup>
-- 
2.18.0

