From 3266ebc9694cf39e6aeb865e98addacdeab6fe26 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 3 Mar 2020 17:51:06 +0000
Subject: [PATCH] Hardcode to netcoreapp3.1 for now

---
 Directory.Build.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 9d86b4fa7..46c5969e4 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -35,7 +35,7 @@
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <LangVersion>8.0</LangVersion>
 
-    <DefaultNetCoreTargetFramework>netcoreapp$(MajorVersion).$(MinorVersion)</DefaultNetCoreTargetFramework>
+    <DefaultNetCoreTargetFramework>netcoreapp3.1</DefaultNetCoreTargetFramework>
 
     <Product>Microsoft .NET Extensions</Product>
     <SignAssembly>true</SignAssembly>
-- 
2.21.1

