From b7ab663e7f70a1043317c87f4e03d141dda45afd Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Fri, 28 Jul 2017 11:06:15 -0700
Subject: [PATCH] Only Build for netstandard

---
 .../Microsoft.Extensions.CommandLineUtils.csproj                     | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/Microsoft.Extensions.CommandLineUtils/Microsoft.Extensions.CommandLineUtils.csproj b/src/Microsoft.Extensions.CommandLineUtils/Microsoft.Extensions.CommandLineUtils.csproj
index 3559126..6c715ff 100644
--- a/src/Microsoft.Extensions.CommandLineUtils/Microsoft.Extensions.CommandLineUtils.csproj
+++ b/src/Microsoft.Extensions.CommandLineUtils/Microsoft.Extensions.CommandLineUtils.csproj
@@ -5,14 +5,14 @@
 Microsoft.Extensions.CommandLineUtils.CommandLineApplication
 Microsoft.Extensions.CommandLineUtils.CommandOption</Description>
     <VersionPrefix>1.0.1</VersionPrefix>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <AssemblyName>Microsoft.Extensions.CommandLineUtils</AssemblyName>
     <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PublicSign>true</PublicSign>
     <PackageId>Microsoft.Extensions.CommandLineUtils</PackageId>
     <PackageTags>commandline;parsing</PackageTags>
     <RepositoryType>git</RepositoryType>
@@ -21,6 +21,7 @@ Microsoft.Extensions.CommandLineUtils.CommandOption</Description>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-- 
1.8.3.1

