From 2830a54c3a3e1f36a15802f1e66e9bcb7a2ffd8a Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 4 Feb 2020 17:34:09 +0000
Subject: [PATCH 05/10] Update empty proj to netcoreapp2.1

---
 eng/empty.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/empty.csproj b/eng/empty.csproj
index 6edde5507fc..b828dfb8225 100644
--- a/eng/empty.csproj
+++ b/eng/empty.csproj
@@ -9,7 +9,7 @@
   -->
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
   <Import Project="versioning.targets" />
-- 
2.18.0

