From a0070894b115dd34313e8c542971d2d90017df74 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 15 May 2020 16:08:26 +0000
Subject: [PATCH 07/15] Update NuGet package build to pass in Configuration

---
 src/coreclr/build.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/coreclr/build.sh b/src/coreclr/build.sh
index 634bcb7af31..1ce18640e17 100755
--- a/src/coreclr/build.sh
+++ b/src/coreclr/build.sh
@@ -302,6 +302,7 @@ generate_NugetPackages()
     "$__RepoRootDir"/eng/common/build.sh -r -b -projects "$__SourceDir"/.nuget/coreclr-packages.proj \
                                        -verbosity minimal -bl:"$__LogsDir/Nuget_$__TargetOS__$__BuildArch__$__BuildType.binlog" \
                                        /p:PortableBuild=true \
+                                       /p:Configuration=$__BuildType \
                                        /p:"__IntermediatesDir=$__IntermediatesDir" /p:"__RootBinDir=$__RootBinDir" /p:"__DoCrossArchBuild=$__CrossBuild" \
                                        $__CommonMSBuildArgs $__UnprocessedBuildArgs
 
-- 
2.17.1

