From 4331afe4a6a6decdc91716e815e286e9dcf7ac64 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 14 Feb 2020 21:02:33 +0000
Subject: [PATCH] Update NuGet package build to pass in Configuration

---
 src/coreclr/build.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/coreclr/build.sh b/src/coreclr/build.sh
index 15e9126..5a6fa32 100755
--- a/src/coreclr/build.sh
+++ b/src/coreclr/build.sh
@@ -302,6 +302,7 @@ generate_NugetPackages()
     "$__RepoRootDir"/eng/common/build.sh -r -b -projects "$__SourceDir"/.nuget/packages.builds \
                                        -verbosity minimal -bl:"$__LogsDir/Nuget_$__TargetOS__$__BuildArch__$__BuildType.binlog" \
                                        /p:PortableBuild=true \
+                                       /p:Configuration=$__BuildType \
                                        /p:"__IntermediatesDir=$__IntermediatesDir" /p:"__RootBinDir=$__RootBinDir" /p:"__DoCrossArchBuild=$__CrossBuild" \
                                        $__CommonMSBuildArgs $__UnprocessedBuildArgs
 
-- 
1.8.3.1

