From c27e2df03be96df63bf6a6173faa54ed5bc5c2ff Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 4 Feb 2020 17:34:09 +0000
Subject: [PATCH] Update empty proj to netcoreapp2.1

---
 eng/empty.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/empty.csproj b/eng/empty.csproj
index 0bd280d..11e8929 100644
--- a/eng/empty.csproj
+++ b/eng/empty.csproj
@@ -9,7 +9,7 @@
   -->
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
 </Project>
-- 
1.8.3.1

