From b993540bec1b2e7af94ab1d3128bf4f30d47f233 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 2 Sep 2020 07:24:30 -0500
Subject: [PATCH 12/13] Add UsingToolNetFrameworkReferenceAssemblies.

---
 src/libraries/restore/runtime/runtime.depproj | 1 +
 1 files changed, 1 insertions(+)

diff --git a/src/libraries/restore/runtime/runtime.depproj b/src/libraries/restore/runtime/runtime.depproj
index 3441fe9399b..36e4bacdef2 100644
--- a/src/libraries/restore/runtime/runtime.depproj
+++ b/src/libraries/restore/runtime/runtime.depproj
@@ -20,6 +20,7 @@
          Exclude here so that when building with the 2.x SDK we don't place it in the test shared framework.
          This can be removed once we have a new SDK -->
     <FileToExclude Include="apphost" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>

   <!-- Setup the testing shared framework host -->
--
2.18.0

