From 3113a8a5880408a0bf99c294afc69a759bb031de Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Tue, 8 Dec 2020 10:16:07 -0600
Subject: [PATCH] Disable building Crossgen2 pack

---
 .../pkg/Microsoft.NETCore.App.Crossgen2.pkgproj | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj b/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
index db13cdde514..71aee46af70 100644
--- a/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
+++ b/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
@@ -10,6 +10,23 @@
     <SkipBuildOnRuntimePackOnlyOS>true</SkipBuildOnRuntimePackOnlyOS>
   </PropertyGroup>
 
+  <!--
+    Building Crossgen2 from source has a few issues:
+
+    Self-contained packages bring down lots of prebuilts. We can disable
+    self-contained, but this only fixes this one problem, and it also makes the
+    source-built crossgen2 different from the Microsoft build and potentially
+    not work in the expected scenarios. See
+    https://github.com/dotnet/source-build/issues/1937
+
+    Even if we build it, there is no mechanism to distribute it in distro
+    packages. This is the same problem as runtime packs. See
+    https://github.com/dotnet/source-build/issues/1568
+  -->
+  <PropertyGroup>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+  </PropertyGroup>
+
   <!--
     Get installer properties, in particular for MSIs.
     Consider porting to central infrastructure: https://github.com/dotnet/runtime/issues/1867
-- 
2.25.4

