From a5316fa11734007708c5eace77a72a5db81632ff Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 27 Oct 2020 17:38:33 +0000
Subject: [PATCH] Update nuget versions to version in SBRP

---
 eng/Versions.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 6fcbcfe25b7..5955f759589 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -127,8 +127,8 @@
     <RefOnlyMicrosoftBuildFrameworkVersion>$(RefOnlyMicrosoftBuildVersion)</RefOnlyMicrosoftBuildFrameworkVersion>
     <RefOnlyMicrosoftBuildTasksCoreVersion>$(RefOnlyMicrosoftBuildVersion)</RefOnlyMicrosoftBuildTasksCoreVersion>
     <RefOnlyMicrosoftBuildUtilitiesCoreVersion>$(RefOnlyMicrosoftBuildVersion)</RefOnlyMicrosoftBuildUtilitiesCoreVersion>
-    <RefOnlyNugetProjectModelVersion>4.9.4</RefOnlyNugetProjectModelVersion>
-    <RefOnlyNugetPackagingVersion>4.9.4</RefOnlyNugetPackagingVersion>
+    <RefOnlyNugetProjectModelVersion>5.1.0</RefOnlyNugetProjectModelVersion>
+    <RefOnlyNugetPackagingVersion>5.1.0</RefOnlyNugetPackagingVersion>
     <!-- Testing -->
     <MicrosoftNETTestSdkVersion>16.8.0-preview-20200730-03</MicrosoftNETTestSdkVersion>
     <MicrosoftDotNetXHarnessTestRunnersXunitVersion>1.0.0-prerelease.20411.1</MicrosoftDotNetXHarnessTestRunnersXunitVersion>
-- 
2.18.4

