From ec3c74b2c90d6a1e6f41e5edcd883a2061efa38f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 7 Aug 2019 09:10:06 -0500
Subject: [PATCH] Fix property name

---
 Directory.Build.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Directory.Build.targets b/Directory.Build.targets
index 657b9547..e1536da5 100644
--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -9,7 +9,7 @@
         <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
         <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
 
-        <PackageReference Update="Microsoft.DotNet.Cli.CommandLine" Version="$(DotNetCliCommandLineVersion)" />
+        <PackageReference Update="Microsoft.DotNet.Cli.CommandLine" Version="$(MicrosoftDotNetCliCommandLinePackageVersion)" />
         <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.8.0" />
         <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
         <PackageReference Update="xunit" Version="2.3.1" />
-- 
2.21.0

