From 162ec70b0f7d8bc7acc4521a507201ecf09e74e2 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 26 Sep 2019 02:58:28 -0500
Subject: [PATCH] Don't call dotnet with no path

---
 eng/common/tools.sh | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 757d5b9e..a8fabf54 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -338,12 +338,6 @@ function MSBuild {
     InitializeBuildTool
     InitializeToolset
 
-    # Work around issues with Azure Artifacts credential provider
-    # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
-
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
     args=( "${args[@]}" "-logger:$logger_path" )
-- 
2.18.0

