From 71baa324a08f12932b29459b544bd2e717dc031a Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 15 May 2017 15:24:45 -0700
Subject: [PATCH] Respect NUGET_PACKAGES if it is set

---
 scripts/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index b253400..458d980 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -67,7 +67,7 @@ done
 #
 TP_ROOT_DIR=$(cd "$(dirname "$0")"; pwd -P)
 TP_TOOLS_DIR="$TP_ROOT_DIR/tools"
-TP_PACKAGES_DIR="$TP_ROOT_DIR/packages"
+TP_PACKAGES_DIR="${NUGET_PACKAGES:-${TP_ROOT_DIR}/packages}"
 TP_OUT_DIR="$TP_ROOT_DIR/artifacts"
 TP_PACKAGE_PROJ_DIR="$TP_ROOT_DIR/src/package/package"
 TP_PACKAGE_NUSPEC_DIR="$TP_ROOT_DIR/src/package/nuspec"
-- 
1.8.3.1

