From c701cbb7c881577581cf44e79022a3354eb999f9 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 22 May 2017 14:10:45 -0700
Subject: [PATCH] Copy prebuilt dotnet from known location

---
 scripts/build.sh | 22 ++++------------------
 1 file changed, 4 insertions(+), 18 deletions(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index 610e63a..a8aea00 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -136,29 +136,15 @@ function usage()
 #
 function install_cli()
 {
-    local failed=false
-    local install_script="$TP_TOOLS_DIR/dotnet-install.sh"
-    local remote_path="https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.sh"
-
     log "Installing dotnet cli..."
     local start=$SECONDS
 
-    # Install the latest version of dotnet-cli
-    curl --retry 10 -sSL --create-dirs -o $install_script $remote_path || failed=true
-    if [ "$failed" = true ]; then
-        error "Failed to download dotnet-install.sh script."
-        return 1
+    if [ ! -e "$TP_TOOLS_DIR/dotnet" ];
+    then
+        mkdir -p "$TP_TOOLS_DIR/dotnet"
     fi
-    chmod u+x $install_script
-
-    log "install_cli: Get the latest dotnet cli toolset..."
-    $install_script --install-dir "$TP_TOOLS_DIR/dotnet" --no-path --channel "master" --version $DOTNET_CLI_VERSION
 
-    # Get netcoreapp1.1 shared components
-    log "install_cli: Get the shared netcoreapp1.0 runtime..."
-    $install_script --install-dir "$TP_TOOLS_DIR/dotnet" --no-path --channel "preview" --version "1.0.5" --shared-runtime
-    log "install_cli: Get the shared netcoreapp1.1 runtime..."
-    $install_script --install-dir "$TP_TOOLS_DIR/dotnet" --no-path --channel "release/1.1.0" --version "1.1.2" --shared-runtime
+    cp -r $DOTNET_TOOL_DIR/* "$TP_TOOLS_DIR/dotnet/"
 
     log "install_cli: Complete. Elapsed $(( SECONDS - start ))s."
     return 0
-- 
1.8.3.1

