From 235237f665ca64c77111270501895fe0d5be8bca Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 22 May 2017 14:10:45 -0700
Subject: [PATCH 2/2] Copy prebuilt dotnet from known location

---
 scripts/build.sh | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index 64cc123..d7150ee 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -72,7 +72,7 @@ TP_SRC_DIR="$TP_ROOT_DIR/src"
 #
 # Dotnet configuration
 #
-# Disable first run since we want to control all package sources 
+# Disable first run since we want to control all package sources
 echo "Setup dotnet configuration."
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 # Dotnet build doesnt support --packages yet. See https://github.com/dotnet/cli/issues/2712
@@ -100,13 +100,7 @@ function installdotnetcli()
     [[ -z "$DOTNET_INSTALL_DIR" ]] && export DOTNET_INSTALL_DIR="$TP_TOOLS_DIR/dotnet"
     [[ -d "$DOTNET_INSTALL_DIR" ]] || mkdir -p $DOTNET_INSTALL_DIR
 
-    DOTNET_INSTALL_PATH="$TP_TOOLS_DIR/dotnet/dotnet"
-    if [[ ! -e $DOTNET_INSTALL_PATH ]]; then
-
-        # Install a stage 0
-        DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh"
-        curl -sSL "$DOTNET_INSTALL_SCRIPT_URL" | bash /dev/stdin  --version $DOTNET_CLI_VERSION --verbose
-    fi
+    cp -r $DOTNET_TOOL_DIR/* $DOTNET_INSTALL_DIR
 
     echo "installdotnetcli: Complete. Elapsed $(( SECONDS - start ))"
 }
-- 
2.11.0

