From 1d5e62d0d67a214806827d93049df2bdd126aa39 Mon Sep 17 00:00:00 2001
From: Sarabjot Singh <sasin@microsoft.com>
Date: Tue, 27 Aug 2019 17:35:20 +0530
Subject: [PATCH] Add the condition for Source build (#2144)

---
 src/datacollector/datacollector.csproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/datacollector/datacollector.csproj b/src/datacollector/datacollector.csproj
index ef76109..2aa91a6 100644
--- a/src/datacollector/datacollector.csproj
+++ b/src/datacollector/datacollector.csproj
@@ -10,6 +10,7 @@
   <PropertyGroup>
     <AssemblyName>datacollector</AssemblyName>
     <TargetFrameworks>netcoreapp2.1;net451</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">netcoreapp2.1</TargetFrameworks>
     <WarningsAsErrors>true</WarningsAsErrors>
     <PlatformTarget Condition="'$(TargetFramework)' == 'net451'">AnyCPU</PlatformTarget>
     <OutputType>Exe</OutputType>
-- 
1.8.3.1

