From 3ce930bd75bd00ba7010f8cbbbfbdfabd9c6e6a1 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 20 Jun 2019 09:59:10 -0500
Subject: [PATCH 3/3] Remove extra dependencies.

---
 src/package/external/external.csproj | 57 ----------------------------
 1 file changed, 57 deletions(-)

diff --git a/src/package/external/external.csproj b/src/package/external/external.csproj
index 78747485..bcba1506 100644
--- a/src/package/external/external.csproj
+++ b/src/package/external/external.csproj
@@ -24,31 +24,6 @@
     <CopyOutputSymbolsToPublishDirectory>false</CopyOutputSymbolsToPublishDirectory>
   </PropertyGroup>
   <ItemGroup>
-    <!-- This csproj restore external tools required for build process -->
-    <PackageReference Include="NuGet.CommandLine">
-      <Version>3.4.3</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="fmdev.xlftool">
-      <Version>0.1.3</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="MicroBuild.Core">
-      <Version>0.2.0</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.Sdk.BuildTasks.14.0">
-      <Version>14.0.12-pre</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="vswhere">
-      <Version>2.0.2</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-	<PackageReference Include="Pdb2Pdb">
-      <Version>1.1.0-beta1-62316-01</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
     <!-- Required for resolution of desktop dependencies in non windows environment.
          We've to set to net46 since the net451 and other packages have assemblies in wrong case.
          E.g. System.XML instead of System.Xml. -->
@@ -56,38 +31,6 @@
       <Version>1.0.1</Version>
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Internal.TestPlatform.Extensions">
-      <Version>$(TestPlatformExternalsVersion)</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.QualityTools">
-      <Version>$(TestPlatformExternalsVersion)</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.QualityTools.DataCollectors">
-      <Version>$(TestPlatformExternalsVersion)</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.CUIT">
-      <Version>$(TestPlatformExternalsVersion)</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Internal.Intellitrace">
-      <Version>16.0.0-preview-20181010-02</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Internal.Dia">
-      <Version>14.0.0</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VSSDK.BuildTools">
-      <Version>15.8.3247</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.Telemetry">
-      <Version>15.6.815-master284DF69C</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
-- 
2.18.0

