From 7db70e74723826deecee1ec0adb21c37d4d0abd8 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 20 Jul 2017 18:03:33 -0700
Subject: [PATCH] Fix typo in build.sh

---
 scripts/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build.sh b/scripts/build.sh
index 40adc77..342be5f 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -262,7 +262,7 @@ function publish_package()
         projectToPackage="${item%%:*}"
         packageOutputPath="${item##*:}"
         log "Package: Publish $projectToPackage"
-        $dotnet publish $projectToPackage --configuration $TPB_Configuration --framework $TPB_TargetFrameworkCore --output $packageOutputPath -v:minimal --p:Version=$TPB_Version -p:CIBuild=$TPB_CIBuild p:LocalizedBuild=$TPB_LocalizedBuild
+        $dotnet publish $projectToPackage --configuration $TPB_Configuration --framework $TPB_TargetFrameworkCore --output $packageOutputPath -v:minimal -p:Version=$TPB_Version -p:CIBuild=$TPB_CIBuild -p:LocalizedBuild=$TPB_LocalizedBuild
     done
 
     # Publish TestHost for netcoreapp1.0 target
-- 
1.8.3.1

