From f78b03fcac3af2a6e5ffc43a53adf27701e7d0ae Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 7 Mar 2019 09:22:05 -0600
Subject: [PATCH] Disable licenseUrl warning.

---
 Directory.Build.props | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index c33640f3..77f2e563 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -7,6 +7,7 @@
     But once we move to a new version, we can remove this property setting, and everything else will work correctly.
     -->
     <UsingMicrosoftNETSdk Condition="'$(MSBuildProjectExtension)' == '.csproj'">true</UsingMicrosoftNETSdk>
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' == 'true'">
@@ -27,4 +28,4 @@
     <BaseIntermediateOutputPath>$(IntermediateOutputPath)</BaseIntermediateOutputPath>
   </PropertyGroup>
 
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

