From d88fe5bf9a48b8b982823a1a7c8a5fea79dcee28 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 22 May 2017 16:32:13 -0700
Subject: [PATCH 3/4] Allow restore without internet sources

---
 dir.props | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/dir.props b/dir.props
index cbb2519..3bb8035 100644
--- a/dir.props
+++ b/dir.props
@@ -118,10 +118,14 @@
     <HostPolicyVersion Condition="'$(UseShippedHostPolicyPackage)' == 'true'">2.0.0</HostPolicyVersion>
   </PropertyGroup>
 
-  <!-- list of nuget package sources passed to dnu -->
   <ItemGroup>
-    <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <DotnetSourceList Include="$(OverridePackageSource)" />
+    <DotnetSourceList Include="$(OverridePackageSourceEx)" />
+  </ItemGroup>
+
+  <!-- list of nuget package sources passed to dnu -->
+  <ItemGroup Condition="'$(ExcludeInternetFeeds)' != 'true'">
+    <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
-- 
1.8.3.1

