From 3653d38bb1bac5b2805d294d7eecd843e053b13c Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Thu, 18 Apr 2019 21:37:14 +0000
Subject: [PATCH] Import dependencies.props after eng/Versions.props

---
 dir.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dir.props b/dir.props
index 20b23c0..8162b3b 100644
--- a/dir.props
+++ b/dir.props
@@ -52,8 +52,8 @@
   <!-- Import Build tools common props file where repo-independent properties are found -->
   <Import Condition="Exists('$(ToolsDir)Build.Common.props')" Project="$(ToolsDir)Build.Common.props" />
 
-  <Import Project="dependencies.props" />
   <Import Project="$(MSBuildThisFileDirectory)eng\Versions.props" />
+  <Import Project="dependencies.props" />
 
   <PropertyGroup Condition="'$(DotNetBuildOffline)' != 'true'">
     <!-- list of nuget package sources passed to NuGet -->
-- 
1.8.3.1

