From 385df662e8b4cf8d5e31aa8de100f0705f80c7ed Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 21 Jun 2019 15:11:41 +0000
Subject: [PATCH] Update Microsoft.Build version to consolidate versions

---
 dependencies.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dependencies.props b/dependencies.props
index 56efa4b..581dc91 100644
--- a/dependencies.props
+++ b/dependencies.props
@@ -32,7 +32,7 @@
   <!-- infrastructure and test only dependencies -->
   <PropertyGroup>
     <NugetProjectModelPackageVersion>4.3.0-preview2-4095</NugetProjectModelPackageVersion>
-    <MicrosoftBuildPackageVersion>15.7.0-preview-000143</MicrosoftBuildPackageVersion>
+    <MicrosoftBuildPackageVersion>15.7.179</MicrosoftBuildPackageVersion>
     <MicrosoftBuildFrameworkPackageVersion>$(MicrosoftBuildPackageVersion)</MicrosoftBuildFrameworkPackageVersion>
     <MicrosoftBuildTasksCorePackageVersion>$(MicrosoftBuildPackageVersion)</MicrosoftBuildTasksCorePackageVersion>
     <MicrosoftBuildUtilitiesCorePackageVersion>$(MicrosoftBuildPackageVersion)</MicrosoftBuildUtilitiesCorePackageVersion>
-- 
1.8.3.1

