From 29acca40d2d5e04c4c080b48e4b383deb1ac99c1 Mon Sep 17 00:00:00 2001
From: Tomas Weinfurt <tweinfurt@yahoo.com>
Date: Wed, 13 Nov 2019 14:07:57 -0800
Subject: [PATCH 7/9] Fix OSGroup on FreeBSD

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index a7e9169c..5089a46d 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -113,6 +113,7 @@
   -->
   <PropertyGroup>
     <OSGroup Condition="'$(OS)'=='Unix' AND Exists('/Applications')">OSX</OSGroup>
+    <OSGroup Condition="'$(OS)'=='Unix' and $([MSBuild]::IsOSPlatform('FREEBSD'))">FreeBSD</OSGroup>
     <OSGroup Condition="'$(OSGroup)' == ''">$(OS)</OSGroup>
   </PropertyGroup>
 
-- 
2.18.0

