From 4cfdc6170926d9eff95b36453c78170c713599f5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sat, 20 Jul 2019 23:17:37 -0500
Subject: [PATCH 7/7] Skip symbol crossgen.

---
 src/dir.props | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/dir.props b/src/dir.props
index cd3eb817..03bd781a 100644
--- a/src/dir.props
+++ b/src/dir.props
@@ -21,9 +21,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <CrossGenSymbolExtension>.map</CrossGenSymbolExtension>
     <CrossGenSymbolExtension Condition="'$(OSGroup)' == 'Windows_NT'">.ni.pdb</CrossGenSymbolExtension>
     <!-- OSX doesn't have crossgen symbols, yet -->
     <CrossGenSymbolExtension Condition="'$(OSGroup)' == 'OSX'"></CrossGenSymbolExtension>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
2.21.0

