From 76a2a98998151ca6db805f3f032cf169f2ec5ef9 Mon Sep 17 00:00:00 2001
From: Omair Majid <omajid@redhat.com>
Date: Thu, 22 Oct 2020 19:07:08 -0400
Subject: [PATCH] Dont build analyzers for source-build

---
 eng/Analyzers.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eng/Analyzers.props b/eng/Analyzers.props
index 438ae20b..97d83e0a 100644
--- a/eng/Analyzers.props
+++ b/eng/Analyzers.props
@@ -1,9 +1,9 @@
 <Project>
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
     <CodeAnalysisRuleset>$(MSBuildThisFileDirectory)CodeAnalysis.ruleset</CodeAnalysisRuleset>
     <EnableAnalyzers>true</EnableAnalyzers>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.0.0-beta3.final" PrivateAssets="all" />
   </ItemGroup>
 </Project>
-- 
2.26.2

