From bd5101b001ed95cb70dde62f53749d0cf9509aa3 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 31 Oct 2018 23:31:42 -0500
Subject: [PATCH] Always use .NET Core ILLink

---
 .../CreateRuntimeRootILLinkDescriptorFile.targets             | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets b/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets
index 967d71cb8d..02c3e297f4 100644
--- a/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets
+++ b/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets
@@ -13,8 +13,8 @@
     <_RexcepFilePath Condition=" '$(_RexcepFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\rexcep.h</_RexcepFilePath>
     <_ILLinkTrimXmlFilePath Condition=" '$(_ILLinkTrimXmlFilePath)' == '' ">$(MSBuildThisFileDirectory)ILLinkTrim.xml</_ILLinkTrimXmlFilePath>
     <_ILLinkTasksToolsDir>$(PackagesDir)/illink.tasks/$(ILLinkTasksPackageVersion)/tools</_ILLinkTasksToolsDir>
-    <_ILLinkTasksDir>$(_ILLinkTasksToolsDir)/net46/</_ILLinkTasksDir>
-    <_ILLinkTasksDir Condition="'$(MSBuildRuntimeType)' == 'Core'">$(_ILLinkTasksToolsDir)/netcoreapp2.0/</_ILLinkTasksDir>
+    <_ILLinkTasksDir>$(_ILLinkTasksToolsDir)/netcoreapp2.0/</_ILLinkTasksDir>
+    <_ILLinkTasksDir Condition="!Exists('$(_ILLinkTasksDir)ILLink.Tasks.dll')">$(_ILLinkTasksToolsDir)/netcoreapp2.0/</_ILLinkTasksDir>
   </PropertyGroup>
 
   <!-- Generates the xml root descriptor file for ILLink. The file contains roots required by the runtime. -->
-- 
2.18.0

