From ac1090a68083e80f92865425b8114a3c65f0568c Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 6 Mar 2019 14:11:30 -0600
Subject: [PATCH] Remove unneeded LangVersion property.

---
 dir.props | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dir.props b/dir.props
index 03f13974e6..e5f39a77b5 100644
--- a/dir.props
+++ b/dir.props
@@ -5,7 +5,6 @@
   <Import Project="dir.common.props" />
 
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
     <UseSharedCompilation>true</UseSharedCompilation>
   </PropertyGroup>
   
-- 
2.18.0

