From f2a9384be10571cc2a4e567605ead6c17e5861d7 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 22 May 2017 15:12:21 -0700
Subject: [PATCH] Unify CLI and BuildTools Versions

---
 DotnetCLIVersion.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DotnetCLIVersion.txt b/DotnetCLIVersion.txt
index 32aae90..170f3fc 100644
--- a/DotnetCLIVersion.txt
+++ b/DotnetCLIVersion.txt
@@ -1 +1 @@
-2.0.0-preview1-005724
\ No newline at end of file
+2.0.0-preview1-005977
-- 
1.8.3.1

