From f3cb2a8bf2ca3fc286438950b9f8af1efb9847d4 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 30 May 2019 09:40:04 -0500
Subject: [PATCH 2/2] Remove deprecation warning.

---
 configurecompiler.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configurecompiler.cmake b/configurecompiler.cmake
index 929dfe5..1631a13 100644
--- a/configurecompiler.cmake
+++ b/configurecompiler.cmake
@@ -451,6 +451,7 @@ if (CLR_CMAKE_PLATFORM_UNIX)
   add_compile_options(-Wno-unused-variable)
   add_compile_options(-Wno-unused-value)
   add_compile_options(-Wno-unused-function)
+  add_compile_options(-Wno-deprecated-register)
 
   #These seem to indicate real issues
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-offsetof")
-- 
1.8.3.1

