From 8c5c1662eda735f032adf0eb4855ad34565e823e Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 7 Jun 2019 18:02:11 -0500
Subject: [PATCH 2/3] Disable new warnings.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 1b43fd479a..65cc76a97e 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -15,5 +15,6 @@
   <PropertyGroup>
     <!-- Enables Strict mode for Roslyn compiler -->
     <Features>strict</Features>
+    <NoWarn>$(NoWarn);CS8604;CS8622</NoWarn>
   </PropertyGroup>
 </Project>
-- 
2.18.0

