From 676022cde4d1f7fac607adaa473c428a42ea07c5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sat, 8 Jun 2019 21:50:54 -0500
Subject: [PATCH 3/3] Remove BinClashLogger.

---
 build-packages.cmd | 1 -
 build.sh           | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/build-packages.cmd b/build-packages.cmd
index 1fa93ebda7..42ea47a27b 100644
--- a/build-packages.cmd
+++ b/build-packages.cmd
@@ -40,7 +40,6 @@ if NOT '%ERRORLEVEL%' == '0' exit /b 1
 
 call %__ProjectDir%/dotnet.cmd msbuild /nologo /verbosity:minimal /clp:Summary /nodeReuse:false^
   /p:__BuildOS=Windows_NT /flp:v=detailed;Append;LogFile=build-packages.log^
-  /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log^
   /p:PortableBuild=true %__ProjectDir%\src\.nuget\packages.builds^
   /p:FilterToOSGroup=Windows_NT %__MSBuildArgs% %unprocessedArgs%
 if NOT [!ERRORLEVEL!]==[0] (
diff --git a/build.sh b/build.sh
index a07f06af68..2aa1e2d5ef 100755
--- a/build.sh
+++ b/build.sh
@@ -256,7 +256,6 @@ build_native()
         if [ $__SkipGenerateVersion == 0 ]; then
             pwd
             "$__ProjectRoot/dotnet.sh" msbuild /nologo /verbosity:minimal /clp:Summary \
-                                       /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll\;LogFile=binclash.log \
                                        /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true \
                                        /p:UsePartialNGENOptimization=false /maxcpucount \
                                        "$__ProjectDir/build.proj" /p:GenerateVersionSourceFile=true /t:GenerateVersionSourceFile /p:NativeVersionSourceFile=$__versionSourceFile \
@@ -455,7 +454,6 @@ build_CoreLib()
     fi
 
     $__ProjectRoot/dotnet.sh msbuild /nologo /verbosity:minimal /clp:Summary \
-                             /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll\;LogFile=binclash.log \
                              /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true \
                              /p:UsePartialNGENOptimization=false /maxcpucount /p:DotNetUseShippingVersions=true /p:ArcadeBuild=true\
                              $__ProjectDir/src/build.proj \
@@ -524,7 +522,6 @@ generate_NugetPackages()
     echo "ROOTFS_DIR is "$ROOTFS_DIR
     # Build the packages
     $__ProjectRoot/dotnet.sh msbuild /nologo /verbosity:minimal /clp:Summary \
-                             /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll\;LogFile=binclash.log \
                              /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true \
                              /p:UsePartialNGENOptimization=false /maxcpucount \
                              $__SourceDir/.nuget/packages.builds \
-- 
2.18.0

