From f26d9c098d6ffde174a8d1b8397116aba56559f2 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Thu, 20 Feb 2020 20:50:05 +0000
Subject: [PATCH 4/4] Remove Helix Dependency

---
 eng/Tools.props | 1 -
 1 file changed, 1 deletion(-)

diff --git a/eng/Tools.props b/eng/Tools.props
index b8c99b0323..eff36ffe08 100644
--- a/eng/Tools.props
+++ b/eng/Tools.props
@@ -9,7 +9,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging" Version="$(MicrosoftDotNetBuildTasksPackagingVersion)" />
-    <PackageDownload Include="Microsoft.DotNet.Helix.Sdk" Version="[$(MicrosoftDotNetHelixSdkVersion)]" />
     <PackageReference Include="Microsoft.NETCore.Platforms" Version="$(MicrosoftNETCorePlatformsVersion)" />
   </ItemGroup>
 </Project>
-- 
2.21.0

