From 820b5105c58f0f5b49b671e532e711f021ca25e1 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 16 Jun 2019 18:39:04 -0500
Subject: [PATCH 5/5] Consolidate MS.NetCore.Platform versions.

---
 dir.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dir.props b/dir.props
index 2792ac731b..866f9362f2 100644
--- a/dir.props
+++ b/dir.props
@@ -137,7 +137,7 @@
     <IsPrerelease>true</IsPrerelease>
 
     <!-- This should be kept in sync with package details in src/.nuget/init/project.json -->
-    <RuntimeIdGraphDefinitionVersion>1.0.2-beta-24224-02</RuntimeIdGraphDefinitionVersion>
+    <RuntimeIdGraphDefinitionVersion>1.1.0</RuntimeIdGraphDefinitionVersion>
     <RuntimeIdGraphDefinitionFile>$(PackagesDir)/microsoft.netcore.platforms/$(RuntimeIdGraphDefinitionVersion)/runtime.json</RuntimeIdGraphDefinitionFile>
 
     <!-- This link should be updated for each release milestone, currently this points to 1.1.0-beta -->
-- 
2.18.0

