From 24bf3fa237519edd672f203c16f2441bc6faf610 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 25 Jul 2019 03:57:02 -0500
Subject: [PATCH 7/7] Add diagnostics to PGO data restore

---
 build.cmd | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build.cmd b/build.cmd
index 94f1f132a2..280a2c51ab 100644
--- a/build.cmd
+++ b/build.cmd
@@ -393,7 +393,7 @@ set PgoDataPackageVersionOutputFile="%__IntermediatesDir%\optdataversion.txt"
 set IbcDataPackageVersionOutputFile="%__IntermediatesDir%\ibcoptdataversion.txt"
 
 REM Parse the optdata package versions out of msbuild so that we can pass them on to CMake
-call "%__ProjectDir%\dotnet.cmd" msbuild "%OptDataProjectFilePath%" /t:DumpPgoDataPackageVersion /nologo %__CommonMSBuildArgs% /p:PgoDataPackageVersionOutputFile="!PgoDataPackageVersionOutputFile!"
+call "%__ProjectDir%\dotnet.cmd" msbuild "%OptDataProjectFilePath%" /t:DumpPgoDataPackageVersion /nologo %__CommonMSBuildArgs% /p:PgoDataPackageVersionOutputFile="!PgoDataPackageVersionOutputFile!" /v:diag
 
  if not !errorlevel! == 0 (
     echo "Failed to get PGO data package version."
@@ -406,7 +406,7 @@ if not exist "!PgoDataPackageVersionOutputFile!" (
 
 set /p __PgoOptDataVersion=<"!PgoDataPackageVersionOutputFile!"
 
-call "%__ProjectDir%\dotnet.cmd" msbuild "%OptDataProjectFilePath%" /t:DumpIbcDataPackageVersion /nologo %__CommonMSBuildArgs% /p:IbcDataPackageVersionOutputFile="!IbcDataPackageVersionOutputFile!"
+call "%__ProjectDir%\dotnet.cmd" msbuild "%OptDataProjectFilePath%" /t:DumpIbcDataPackageVersion /nologo %__CommonMSBuildArgs% /p:IbcDataPackageVersionOutputFile="!IbcDataPackageVersionOutputFile!" /v:diag
 
  if not !errorlevel! == 0 (
     echo "Failed to get IBC data package version."
-- 
2.18.0

