From 6e32f0e1b22fb35e2abd9c08fe43a6f5a6b28c24 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 7 Mar 2019 12:14:30 -0600
Subject: [PATCH] Add type annotation to F#?

---
 src/utils/reshapedreflection.fs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/reshapedreflection.fs b/src/utils/reshapedreflection.fs
index b1f0aeca8..33e9b0c73 100644
--- a/src/utils/reshapedreflection.fs
+++ b/src/utils/reshapedreflection.fs
@@ -360,7 +360,7 @@ module internal ReflectionAdapters =
             this.GetType().GetMethod(methodName, argTypes, BindingFlags.Public)
 
     type System.Char with
-        static member GetUnicodeCategory(c) = System.Globalization.CharUnicodeInfo.GetUnicodeCategory(c)
+        static member GetUnicodeCategory(c) = System.Globalization.CharUnicodeInfo.GetUnicodeCategory(c : char)
 
     type System.Reflection.Module with
         member this.MetadataToken =
-- 
2.18.0

