From cb14a9e1df7b8a5e681b36acb5454e28a7ea5b75 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 7 Mar 2019 13:15:57 -0600
Subject: [PATCH] netcoreapp2.1 cascading.

---
 src/buildfromsource/Fsi/Fsi.fsproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/buildfromsource/Fsi/Fsi.fsproj b/src/buildfromsource/Fsi/Fsi.fsproj
index 69daac776..83061c30a 100644
--- a/src/buildfromsource/Fsi/Fsi.fsproj
+++ b/src/buildfromsource/Fsi/Fsi.fsproj
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <TargetExt>.exe</TargetExt>
     <AssemblyName>fsi</AssemblyName>
     <NoWarn>$(NoWarn);45;55;62;75;1204</NoWarn>
-- 
2.18.0

