From f7349efa63672db8c08a03ffd15124e69c4880f5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 3 Sep 2019 12:17:10 -0500
Subject: [PATCH 3/3] Remove some old feeds.

---
 NuGet.config       | 3 ---
 eng/Versions.props | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/NuGet.config b/NuGet.config
index a8664a73c..0a2654257 100644
--- a/NuGet.config
+++ b/NuGet.config
@@ -18,10 +18,7 @@
     <add key="interactive-window" value="https://dotnet.myget.org/F/interactive-window/api/v3/index.json" />
     <add key="vs-devcore" value="https://myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
-    <add key="vssdk" value="https://vside.myget.org/F/vssdk/api/v3/index.json" />
-    <add key="vs-impl" value="https://vside.myget.org/F/vs-impl/api/v3/index.json" />
     <add key="roslyn_concord" value="https://myget.org/F/roslyn_concord/api/v3/index.json" />
-    <add key="devcore" value="https://vside.myget.org/F/devcore/api/v3/index.json" />
   </packageSources>
   <disabledPackageSources>
     <clear />
diff --git a/eng/Versions.props b/eng/Versions.props
index ac79040a6..449e639f5 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -59,10 +59,7 @@
       https://dotnet.myget.org/F/interactive-window/api/v3/index.json;
       https://myget.org/F/vs-devcore/api/v3/index.json;
       https://myget.org/F/vs-editor/api/v3/index.json;
-      https://vside.myget.org/F/vssdk/api/v3/index.json;
-      https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://myget.org/F/roslyn_concord/api/v3/index.json;
-      https://vside.myget.org/F/devcore/api/v3/index.json;
     </RestoreSources>
     <!-- version numbers from files -->
     <RoslynVersion>$([System.IO.File]::ReadAllText('$(MSBuildThisFileDirectory)..\RoslynPackageVersion.txt').Trim())</RoslynVersion>
-- 
2.21.0

