From 1110d5105afe8387ad899727aab2c8c5774ba1a5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 7 Mar 2019 14:22:27 -0600
Subject: [PATCH] Fix casing in project.

---
 src/buildfromsource/Fsc/Fsc.fsproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/buildfromsource/Fsc/Fsc.fsproj b/src/buildfromsource/Fsc/Fsc.fsproj
index c5178aaca..df3e6d691 100644
--- a/src/buildfromsource/Fsc/Fsc.fsproj
+++ b/src/buildfromsource/Fsc/Fsc.fsproj
@@ -20,7 +20,7 @@
     <Compile Include="$(FSharpSourcesRoot)\fsharp\fscmain.fs">
       <Link>fscmain.fs</Link>
     </Compile>
-    <None Include="$(FSharpSourcesRoot)\fsharp\Fsc\app.config" />
+    <None Include="$(FSharpSourcesRoot)\fsharp\fsc\app.config" />
     <None Include="..\..\windows\default.win32manifest">
       <Link>default.win32manifest</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-- 
2.18.0

