From 2ffee3994c62b1b7ea5141f83298d2475142271c Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 9 Dec 2019 12:38:25 -0600
Subject: [PATCH 5/5] Fix bad binlog path

---
 eng/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/build.sh b/eng/build.sh
index 70db3e848..d95a70fd9 100755
--- a/eng/build.sh
+++ b/eng/build.sh
@@ -192,7 +192,7 @@ function BuildSolution {
   
   local bl=""
   if [[ "$binary_log" = true ]]; then
-    bl="/bl:\"$log_dir/Build.binlog\""
+    bl="/bl"
   fi
   
   local projects="$repo_root/$solution" 
-- 
2.18.0

