From 77ec6651340f0b1dfd477c7868b533b3fde6d522 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 10 Mar 2019 00:00:32 -0600
Subject: [PATCH] Don't reset RestoreSources when offline.

---
 build/targets/PackageVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/targets/PackageVersions.props b/build/targets/PackageVersions.props
index 97165c5fe..8ecdeb50a 100644
--- a/build/targets/PackageVersions.props
+++ b/build/targets/PackageVersions.props
@@ -3,7 +3,7 @@
   <PropertyGroup>
 
     <!-- default package sources -->
-    <RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
       https://www.myget.org/F/fsharp-daily/api/v3/index.json;
       https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json;
-- 
2.18.0

