From e78b05d8d19744cfe25ad996298a40253f59863e Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Thu, 10 Aug 2017 20:11:00 -0400
Subject: [PATCH 1/1] Fix copy targets to nupkg

---
 src/FSharpSource.BuildFromSource.targets | 20 ++++++++------------
 1 file changed, 8 insertions(+), 12 deletions(-)

diff --git a/src/FSharpSource.BuildFromSource.targets b/src/FSharpSource.BuildFromSource.targets
index 82c9b63..5e50595 100644
--- a/src/FSharpSource.BuildFromSource.targets
+++ b/src/FSharpSource.BuildFromSource.targets
@@ -39,18 +39,14 @@
           BeforeTargets="Compile"
           Inputs="@(CopyAndSubstituteText)"
           Outputs="@(CopyAndSubstituteText->'$(OutDir)%(TargetFilename)')" >
-
-    <PropertyGroup>
-      <FileText>
-        $([System.Text.RegularExpressions.Regex]::Replace($([System.IO.File]::ReadAllText("%(CopyAndSubstituteText.FullPath)")), "%(CopyAndSubstituteText.Pattern1)", "%(CopyAndSubstituteText.Replacement1)"))
-      </FileText>
-
-      <FileText Condition = "'%(CopyAndSubstituteText.Pattern2)' != ''">
-        $([System.Text.RegularExpressions.Regex]::Replace($(FileText), "%(CopyAndSubstituteText.Pattern2)", "%(CopyAndSubstituteText.Replacement2)"))
-      </FileText>
-    </PropertyGroup>
-    <WriteLinesToFile File="$(OutDir)%(CopyAndSubstituteText.TargetFilename)" Lines="$(FileText)" Overwrite="true" />
-  </Target>
+    <ItemGroup>
+      <FileTextMap Include="%(CopyAndSubstituteText.TargetFilename)">
+        <FileText>$([System.Text.RegularExpressions.Regex]::Replace($([System.IO.File]::ReadAllText("%(CopyAndSubstituteText.FullPath)")), "%(CopyAndSubstituteText.Pattern1)", "%(CopyAndSubstituteText.Replacement1)"))</FileText>
+        <FileText Condition = "'$(CurrentPattern2)' != ''">$([System.Text.RegularExpressions.Regex]::Replace($(FileText), "%(CopyAndSubstituteText.Pattern2)", "%(CopyAndSubstituteText.Replacement2)"))</FileText>
+      </FileTextMap>
+    </ItemGroup>
+    <WriteLinesToFile File="$(OutDir)%(FileTextMap.Identity)" Lines="%(FileTextMap.FileText)" Overwrite="true" />
+ </Target>
 
   <!--
   =========================================================================
-- 
1.8.3.1

