From 90b2ea23dd4aa2c381fa9023d704ff035fff6a5b Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 2 Oct 2019 10:54:19 -0500
Subject: [PATCH] Debugging patch

---
 .../src/NuGetPack.cs                          | 33 +++++++++++++++++++
 1 file changed, 33 insertions(+)

diff --git a/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs b/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs
index 5fb7c2e7..29faa162 100644
--- a/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs
+++ b/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs
@@ -12,6 +12,8 @@
 using System.Linq;
 using NuGet.Common;
 
+using System.Diagnostics;
+
 namespace Microsoft.DotNet.Build.Tasks.Packaging
 {
     public class NuGetPack : BuildTask
@@ -241,16 +243,47 @@ private string GetPackageOutputPath(string nuspecPath, Manifest manifest, bool i
             string nupkgExtension = isSymbolsPackage ? _symbolsPackageExtension : _packageExtension;
             return Path.Combine(nupkgOutputDirectory, $"{id}.{version}{nupkgExtension}");
         }
+private string GetInode(string path) {
+  var psi = new ProcessStartInfo();
+  psi.Arguments = $"--format='%i' {path}";
+  psi.FileName = "stat";
+  psi.RedirectStandardOutput = true;
+  var p = Process.Start(psi);
+  var output = p.StandardOutput.ReadToEnd();
+  p.WaitForExit();
+  return output.Trim();
+}
 
         public void Pack(string nuspecPath, string nupkgPath, Manifest manifest, bool packSymbols)
         {
             bool creatingSymbolsPackage = packSymbols && (Path.GetExtension(nupkgPath) == _symbolsPackageExtension);
+            Log.LogWarning($"[{DateTimeOffset.Now}] starting NuGetPack.Pack: nuspecPath={nuspecPath}, nupkgPath={nupkgPath}, packSymbols={packSymbols}");
+            Log.LogWarning($"[{DateTimeOffset.Now}] OutputDirectory={OutputDirectory}, BaseDirectory={BaseDirectory ?? "<null>"}, PackageVersion={PackageVersion ?? "<null>"}");
+            Log.LogWarning($"[{DateTimeOffset.Now}] ExcludeEmptyDirectories={ExcludeEmptyDirectories}, CreateSymbolPackage={CreateSymbolPackage}, IncludeSymbolsInPackage={IncludeSymbolsInPackage}");
+            Log.LogWarning($"[{DateTimeOffset.Now}] CreatePackedPackage={CreatePackedPackage}, SymbolPackageOutputDirectory={SymbolPackageOutputDirectory ?? "<null>"}, PackedPackageNamePrefix={PackedPackageNamePrefix ?? "<null>"}");
+
             try
             {
                 PackageBuilder builder = new PackageBuilder();
 
                 string baseDirectoryPath = (string.IsNullOrEmpty(BaseDirectory)) ? Path.GetDirectoryName(nuspecPath) : BaseDirectory;
                 builder.Populate(manifest.Metadata);
+                Log.LogWarning($"[{DateTimeOffset.Now}] about to populate files: baseDirectory={baseDirectoryPath}");
+                foreach (var f in manifest.Files)
+                {
+                    Log.LogWarning($"[{DateTimeOffset.Now}] file: Source={f.Source}, Target={f.Target}, Exclude={f.Exclude}");
+                    var s = f.Source;
+                    while (s != "/")
+                    {
+                        Log.LogWarning($"[{DateTimeOffset.Now}] File {s} exists: {File.Exists(s)}");
+                        Log.LogWarning($"[{DateTimeOffset.Now}] Directory {s} exists: {Directory.Exists(s)}");
+                        if (Directory.Exists(s))
+                        {
+                            Log.LogWarning($"[{DateTimeOffset.Now}] Files in {s} (inode {GetInode(s)}): {string.Join(",", Directory.EnumerateFiles(s))}");
+                        }
+                        s = Directory.GetParent(s).FullName;
+                    }
+                }
                 builder.PopulateFiles(baseDirectoryPath, manifest.Files);
 
                 if (creatingSymbolsPackage)
-- 
2.18.0

