From 94291517876d0cfb57eeb5bd34f8f26d8ef43ffe Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 18 Sep 2019 21:50:06 -0500
Subject: [PATCH 2/2] Build CoreFxTesting

---
 .../Microsoft.DotNet.CoreFxTesting.csproj                        | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Microsoft.DotNet.CoreFxTesting/Microsoft.DotNet.CoreFxTesting.csproj b/src/Microsoft.DotNet.CoreFxTesting/Microsoft.DotNet.CoreFxTesting.csproj
index 78d8c94c..8bd6963a 100644
--- a/src/Microsoft.DotNet.CoreFxTesting/Microsoft.DotNet.CoreFxTesting.csproj
+++ b/src/Microsoft.DotNet.CoreFxTesting/Microsoft.DotNet.CoreFxTesting.csproj
@@ -4,7 +4,6 @@
     <TargetFrameworks>netcoreapp2.1;net472</TargetFrameworks>
     <Description>This package provides support for running tests inside CoreFx.</Description>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.18.0

