From b85de43a04b54a267ac4fa8dd70047b35e482676 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 29 Aug 2019 11:20:06 -0500
Subject: [PATCH 2/2] Remove adding fake dependencies for CoreFX partial
 packages.

---
 .../src/build/Packaging.targets                            | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets b/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets
index a3a0bd85..06d45eff 100644
--- a/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets
+++ b/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets
@@ -590,13 +590,6 @@
       <_harvestFile Include="@(File)" Condition="'%(File.HarvestDependencies)' == 'true' and '%(File.Extension)' == '.dll'" />
       <_missingHarvestFile Include="@(_harvestFile)" Condition="!Exists('%(FullPath)')" />
       <_harvestFile Remove="@(_missingHarvestFile)" Condition="'$(AllowPartialPackages)' == 'true'" />
-
-      <!-- add a fake dependency to represent the dependencies of any missing file
-           this will prevent the package from installing on that platform. -->
-      <FilePackageDependency Include="Unavailable" Condition="'$(AllowPartialPackages)' == 'true' AND '@(_missingHarvestFile)' != '' AND '$(_TargetFramework)' != ''">
-        <TargetFramework>$(_TargetFramework)</TargetFramework>
-        <Version>0.0.0</Version>
-      </FilePackageDependency>
     </ItemGroup>
 
     <!-- Generate package references based on assembly dependencies -->
-- 
2.18.0

