From dbaea15cd264f465bb09547553cfb94c24eaa655 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Fri, 26 Oct 2018 15:10:31 -0500
Subject: [PATCH] Remove projects that die on DotNetBuildFromSource

---
 Arcade.sln | 70 ------------------------------------------------------
 1 file changed, 70 deletions(-)

diff --git a/Arcade.sln b/Arcade.sln
index 3a88069..c01fc81 100644
--- a/Arcade.sln
+++ b/Arcade.sln
@@ -72,8 +72,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Darc", "sr
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Maestro.MergePolicies", "src\Maestro\Maestro.MergePolicies\Maestro.MergePolicies.csproj", "{37D70EEA-9621-44EB-921A-5D303917F851}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.GenFacades", "src\Microsoft.DotNet.GenFacades\Microsoft.DotNet.GenFacades.csproj", "{B6778A84-3330-4F2E-9D26-316C78A4BCC2}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Cci.Extensions", "src\Microsoft.Cci.Extensions\Microsoft.Cci.Extensions.csproj", "{BCBFB7E1-B202-4C88-AB9F-290B13F309FD}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.DotNet.SwaggerGenerator", "Microsoft.DotNet.SwaggerGenerator", "{BB99CE55-AD68-484A-9474-B8BACED247AB}"
@@ -84,18 +82,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SwaggerGen
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SwaggerGenerator.MSBuild", "src\Microsoft.DotNet.SwaggerGenerator\Microsoft.DotNet.SwaggerGenerator.MSBuild\Microsoft.DotNet.SwaggerGenerator.MSBuild.csproj", "{4FA6B420-53A3-4956-9C02-8649A720A9F5}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.GenAPI", "src\Microsoft.DotNet.GenAPI\Microsoft.DotNet.GenAPI.csproj", "{9427265E-C224-4B40-8157-F05529D084D3}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.ApiCompat", "src\Microsoft.DotNet.ApiCompat\Microsoft.DotNet.ApiCompat.csproj", "{9E406363-CFB1-4851-AEE9-9DEF98A7F814}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VersionManager.Tests", "src\Maestro\tests\VersionManager.Tests\VersionManager.Tests.csproj", "{2CCBE568-1846-48BF-9294-3CD25F66C098}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SignCheck", "SignCheck", "{A704E5B2-86A4-4D4F-902D-C10EA18C12DF}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SignCheck", "src\SignCheck\SignCheck\Microsoft.DotNet.SignCheck.csproj", "{AF298985-511F-476A-8F62-79908BE8DF01}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SignCheckLibrary", "src\SignCheck\Microsoft.SignCheck\Microsoft.DotNet.SignCheckLibrary.csproj", "{39B69A56-43E9-480D-9A60-8E0917795FE8}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WinShimmer", "src\WinShimmer\WinShimmer.csproj", "{39C3FC42-21E8-40BF-AE3F-CB9C4CC226D7}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.Packaging", "src\Microsoft.DotNet.Build.Tasks.Packaging\src\Microsoft.DotNet.Build.Tasks.Packaging.csproj", "{EFC8C541-1DC4-40DA-B6E5-FBF44F6DD8BC}"
@@ -502,18 +490,6 @@ Global
 		{37D70EEA-9621-44EB-921A-5D303917F851}.Release|x64.Build.0 = Release|Any CPU
 		{37D70EEA-9621-44EB-921A-5D303917F851}.Release|x86.ActiveCfg = Release|Any CPU
 		{37D70EEA-9621-44EB-921A-5D303917F851}.Release|x86.Build.0 = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|x64.Build.0 = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|x86.Build.0 = Debug|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|Any CPU.Build.0 = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|x64.ActiveCfg = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|x64.Build.0 = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|x86.ActiveCfg = Release|Any CPU
-		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Release|x86.Build.0 = Release|Any CPU
 		{BCBFB7E1-B202-4C88-AB9F-290B13F309FD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{BCBFB7E1-B202-4C88-AB9F-290B13F309FD}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{BCBFB7E1-B202-4C88-AB9F-290B13F309FD}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -562,30 +538,6 @@ Global
 		{4FA6B420-53A3-4956-9C02-8649A720A9F5}.Release|x64.Build.0 = Release|Any CPU
 		{4FA6B420-53A3-4956-9C02-8649A720A9F5}.Release|x86.ActiveCfg = Release|Any CPU
 		{4FA6B420-53A3-4956-9C02-8649A720A9F5}.Release|x86.Build.0 = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|x64.Build.0 = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Debug|x86.Build.0 = Debug|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|Any CPU.Build.0 = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|x64.ActiveCfg = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|x64.Build.0 = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|x86.ActiveCfg = Release|Any CPU
-		{9427265E-C224-4B40-8157-F05529D084D3}.Release|x86.Build.0 = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|x64.Build.0 = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Debug|x86.Build.0 = Debug|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|Any CPU.Build.0 = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|x64.ActiveCfg = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|x64.Build.0 = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|x86.ActiveCfg = Release|Any CPU
-		{9E406363-CFB1-4851-AEE9-9DEF98A7F814}.Release|x86.Build.0 = Release|Any CPU
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -598,26 +550,6 @@ Global
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Release|x64.Build.0 = Release|Any CPU
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Release|x86.ActiveCfg = Release|Any CPU
 		{2CCBE568-1846-48BF-9294-3CD25F66C098}.Release|x86.Build.0 = Release|Any CPU
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Debug|Any CPU.Build.0 = Debug|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Debug|x64.ActiveCfg = Debug|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Debug|x86.ActiveCfg = Debug|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Debug|x86.Build.0 = Debug|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Release|Any CPU.ActiveCfg = Release|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Release|Any CPU.Build.0 = Release|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Release|x64.ActiveCfg = Release|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Release|x86.ActiveCfg = Release|x86
-		{AF298985-511F-476A-8F62-79908BE8DF01}.Release|x86.Build.0 = Release|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Debug|Any CPU.Build.0 = Debug|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Debug|x64.ActiveCfg = Debug|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Debug|x86.ActiveCfg = Debug|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Debug|x86.Build.0 = Debug|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Release|Any CPU.ActiveCfg = Release|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Release|Any CPU.Build.0 = Release|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Release|x64.ActiveCfg = Release|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Release|x86.ActiveCfg = Release|x86
-		{39B69A56-43E9-480D-9A60-8E0917795FE8}.Release|x86.Build.0 = Release|x86
 		{39C3FC42-21E8-40BF-AE3F-CB9C4CC226D7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{39C3FC42-21E8-40BF-AE3F-CB9C4CC226D7}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{39C3FC42-21E8-40BF-AE3F-CB9C4CC226D7}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -709,8 +641,6 @@ Global
 		{697EB593-3277-4C43-B8F3-C578E21B051B} = {BB99CE55-AD68-484A-9474-B8BACED247AB}
 		{4FA6B420-53A3-4956-9C02-8649A720A9F5} = {BB99CE55-AD68-484A-9474-B8BACED247AB}
 		{2CCBE568-1846-48BF-9294-3CD25F66C098} = {37184B58-835A-4D51-AA58-250430D283AA}
-		{AF298985-511F-476A-8F62-79908BE8DF01} = {A704E5B2-86A4-4D4F-902D-C10EA18C12DF}
-		{39B69A56-43E9-480D-9A60-8E0917795FE8} = {A704E5B2-86A4-4D4F-902D-C10EA18C12DF}
 		{F23ED302-7451-4D40-85C3-FACFCF884661} = {C53DD924-C212-49EA-9BC4-1827421361EF}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
-- 
2.17.1.windows.2

