From 47664d5f40853e753eac16764c77b3d8500e7db8 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 2 May 2019 01:32:35 -0500
Subject: [PATCH] Remove test projects.

---
 Arcade.sln | 119 -----------------------------------------------------
 1 file changed, 119 deletions(-)

diff --git a/Arcade.sln b/Arcade.sln
index 8e5357db..b571d36b 100644
--- a/Arcade.sln
+++ b/Arcade.sln
@@ -24,8 +24,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.ServiceFab
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Github.IssueLabeler", "src\Microsoft.DotNet.Github.IssueLabeler\Microsoft.DotNet.Github.IssueLabeler.csproj", "{A32C09E4-C334-4614-B40F-DEDF352C820E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SignTool.Tests", "src\Microsoft.DotNet.SignTool.Tests\Microsoft.DotNet.SignTool.Tests.csproj", "{25ACF626-1C89-4A32-A434-E3DA17759631}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.GenFacades", "src\Microsoft.DotNet.GenFacades\Microsoft.DotNet.GenFacades.csproj", "{B6778A84-3330-4F2E-9D26-316C78A4BCC2}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Cci.Extensions", "src\Microsoft.Cci.Extensions\Microsoft.Cci.Extensions.csproj", "{BCBFB7E1-B202-4C88-AB9F-290B13F309FD}"
@@ -52,24 +50,14 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WinShimmer", "src\WinShimme
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.Packaging", "src\Microsoft.DotNet.Build.Tasks.Packaging\src\Microsoft.DotNet.Build.Tasks.Packaging.csproj", "{EFC8C541-1DC4-40DA-B6E5-FBF44F6DD8BC}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.Packaging.Tests", "src\Microsoft.DotNet.Build.Tasks.Packaging\tests\Microsoft.DotNet.Build.Tasks.Packaging.Tests.csproj", "{F23ED302-7451-4D40-85C3-FACFCF884661}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{C53DD924-C212-49EA-9BC4-1827421361EF}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.CodeAnalysis", "src\Microsoft.DotNet.CodeAnalysis\Microsoft.DotNet.CodeAnalysis.csproj", "{AA00BFF4-D182-475F-85B9-BF4924F4D794}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HelixPoolProvider", "HelixPoolProvider", "{86AEE6AF-16D7-47FD-8D3E-6B87B249208D}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.HelixPoolProvider", "src\HelixPoolProvider\HelixPoolProvider\Microsoft.DotNet.HelixPoolProvider.csproj", "{5ED29E38-A65D-4123-BFED-799ADA885F08}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.CoreFxTesting", "src\Microsoft.DotNet.CoreFxTesting\Microsoft.DotNet.CoreFxTesting.csproj", "{73389197-EBB2-4FB7-B599-9573971A7197}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.Configuration", "src\Microsoft.DotNet.Build.Tasks.Configuration\src\Microsoft.DotNet.Build.Tasks.Configuration.csproj", "{357E3759-EAF2-4F9C-918E-64F801DE3B57}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.Configuration.Tests", "src\Microsoft.DotNet.Build.Tasks.Configuration\tests\Microsoft.DotNet.Build.Tasks.Configuration.Tests.csproj", "{8693536F-E31B-4841-8D82-7162E9D753D9}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Arcade.Sdk.Tests", "src\Microsoft.DotNet.Arcade.Sdk.Tests\Microsoft.DotNet.Arcade.Sdk.Tests.csproj", "{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Helix.Client", "src\Microsoft.DotNet.Helix\Client\CSharp\Microsoft.DotNet.Helix.Client.csproj", "{DC0D0EE5-1C0C-4D3E-BA0B-D6A59717DA94}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.SourceRewriter", "src\Microsoft.DotNet.SourceRewriter\Microsoft.DotNet.SourceRewriter.csproj", "{464D41D0-D1C4-4D3C-BD43-28A2B4260EFF}"
@@ -80,12 +68,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "VisualStudio", "VisualStudi
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.VisualStudio", "src\Microsoft.DotNet.Build.Tasks.VisualStudio\Microsoft.DotNet.Build.Tasks.VisualStudio.csproj", "{3CCA947D-B466-4481-BC78-43CE98A8A3A7}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Build.Tasks.VisualStudio.Tests", "src\Microsoft.DotNet.Build.Tasks.VisualStudio.Tests\Microsoft.DotNet.Build.Tasks.VisualStudio.Tests.csproj", "{86D901E6-C054-445B-92EC-E267EBB16278}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.VersionTools", "src\Microsoft.DotNet.VersionTools\lib\Microsoft.DotNet.VersionTools.csproj", "{465F7F95-1774-40BE-8387-62C39BBD7141}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.VersionTools.Tests", "src\Microsoft.DotNet.VersionTools\tests\Microsoft.DotNet.VersionTools.Tests.csproj", "{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.VersionTools.Tasks", "src\Microsoft.DotNet.VersionTools\tasks\Microsoft.DotNet.VersionTools.Tasks.csproj", "{0A6A6D3D-DE7A-4BC2-9799-9C77E4CE87FB}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DotNet.Git.IssueManager", "src\Microsoft.DotNet.Git.IssueManager\src\Microsoft.DotNet.Git.IssueManager.csproj", "{20F768D8-D724-4889-85AE-2D46188E731F}"
@@ -94,8 +78,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.RemoteExec
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.RemoteExecutorHost", "src\Microsoft.DotNet.RemoteExecutor\src\Microsoft.DotNet.RemoteExecutorHost\Microsoft.DotNet.RemoteExecutorHost.csproj", "{5588033C-A6F5-4824-8E58-A369F644CDF8}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.XUnitExtensions.Tests", "src\Microsoft.DotNet.XUnitExtensions\tests\Microsoft.DotNet.XUnitExtensions.Tests.csproj", "{036A1269-EA1B-457A-8447-D90AEF80BAF9}"
-EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -226,18 +208,6 @@ Global
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Release|x64.Build.0 = Release|Any CPU
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Release|x86.ActiveCfg = Release|Any CPU
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Release|x86.Build.0 = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|x64.Build.0 = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Debug|x86.Build.0 = Debug|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|Any CPU.Build.0 = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|x64.ActiveCfg = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|x64.Build.0 = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|x86.ActiveCfg = Release|Any CPU
-		{25ACF626-1C89-4A32-A434-E3DA17759631}.Release|x86.Build.0 = Release|Any CPU
 		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{B6778A84-3330-4F2E-9D26-316C78A4BCC2}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -366,18 +336,6 @@ Global
 		{EFC8C541-1DC4-40DA-B6E5-FBF44F6DD8BC}.Release|x64.Build.0 = Release|Any CPU
 		{EFC8C541-1DC4-40DA-B6E5-FBF44F6DD8BC}.Release|x86.ActiveCfg = Release|Any CPU
 		{EFC8C541-1DC4-40DA-B6E5-FBF44F6DD8BC}.Release|x86.Build.0 = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|x64.Build.0 = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Debug|x86.Build.0 = Debug|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|Any CPU.Build.0 = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|x64.ActiveCfg = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|x64.Build.0 = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|x86.ActiveCfg = Release|Any CPU
-		{F23ED302-7451-4D40-85C3-FACFCF884661}.Release|x86.Build.0 = Release|Any CPU
 		{AA00BFF4-D182-475F-85B9-BF4924F4D794}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{AA00BFF4-D182-475F-85B9-BF4924F4D794}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{AA00BFF4-D182-475F-85B9-BF4924F4D794}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -398,18 +356,6 @@ Global
 		{5ED29E38-A65D-4123-BFED-799ADA885F08}.Release|Any CPU.Build.0 = Release|Any CPU
 		{5ED29E38-A65D-4123-BFED-799ADA885F08}.Release|x64.ActiveCfg = Release|Any CPU
 		{5ED29E38-A65D-4123-BFED-799ADA885F08}.Release|x86.ActiveCfg = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|x64.Build.0 = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Debug|x86.Build.0 = Debug|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|Any CPU.Build.0 = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|x64.ActiveCfg = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|x64.Build.0 = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|x86.ActiveCfg = Release|Any CPU
-		{73389197-EBB2-4FB7-B599-9573971A7197}.Release|x86.Build.0 = Release|Any CPU
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -422,30 +368,6 @@ Global
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Release|x64.Build.0 = Release|Any CPU
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Release|x86.ActiveCfg = Release|Any CPU
 		{357E3759-EAF2-4F9C-918E-64F801DE3B57}.Release|x86.Build.0 = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|x64.Build.0 = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Debug|x86.Build.0 = Debug|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|Any CPU.Build.0 = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|x64.ActiveCfg = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|x64.Build.0 = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|x86.ActiveCfg = Release|Any CPU
-		{8693536F-E31B-4841-8D82-7162E9D753D9}.Release|x86.Build.0 = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|x64.Build.0 = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Debug|x86.Build.0 = Debug|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|Any CPU.Build.0 = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|x64.ActiveCfg = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|x64.Build.0 = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|x86.ActiveCfg = Release|Any CPU
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8}.Release|x86.Build.0 = Release|Any CPU
 		{DC0D0EE5-1C0C-4D3E-BA0B-D6A59717DA94}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{DC0D0EE5-1C0C-4D3E-BA0B-D6A59717DA94}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{DC0D0EE5-1C0C-4D3E-BA0B-D6A59717DA94}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -494,18 +416,6 @@ Global
 		{3CCA947D-B466-4481-BC78-43CE98A8A3A7}.Release|x64.Build.0 = Release|Any CPU
 		{3CCA947D-B466-4481-BC78-43CE98A8A3A7}.Release|x86.ActiveCfg = Release|Any CPU
 		{3CCA947D-B466-4481-BC78-43CE98A8A3A7}.Release|x86.Build.0 = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|x64.Build.0 = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Debug|x86.Build.0 = Debug|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|Any CPU.Build.0 = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|x64.ActiveCfg = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|x64.Build.0 = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|x86.ActiveCfg = Release|Any CPU
-		{86D901E6-C054-445B-92EC-E267EBB16278}.Release|x86.Build.0 = Release|Any CPU
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -518,18 +428,6 @@ Global
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Release|x64.Build.0 = Release|Any CPU
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Release|x86.ActiveCfg = Release|Any CPU
 		{465F7F95-1774-40BE-8387-62C39BBD7141}.Release|x86.Build.0 = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|x64.Build.0 = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Debug|x86.Build.0 = Debug|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|Any CPU.Build.0 = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|x64.ActiveCfg = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|x64.Build.0 = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|x86.ActiveCfg = Release|Any CPU
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C}.Release|x86.Build.0 = Release|Any CPU
 		{0A6A6D3D-DE7A-4BC2-9799-9C77E4CE87FB}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{0A6A6D3D-DE7A-4BC2-9799-9C77E4CE87FB}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{0A6A6D3D-DE7A-4BC2-9799-9C77E4CE87FB}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -578,18 +476,6 @@ Global
 		{5588033C-A6F5-4824-8E58-A369F644CDF8}.Release|x64.Build.0 = Release|Any CPU
 		{5588033C-A6F5-4824-8E58-A369F644CDF8}.Release|x86.ActiveCfg = Release|Any CPU
 		{5588033C-A6F5-4824-8E58-A369F644CDF8}.Release|x86.Build.0 = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|x64.Build.0 = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Debug|x86.Build.0 = Debug|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|Any CPU.Build.0 = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|x64.ActiveCfg = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|x64.Build.0 = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|x86.ActiveCfg = Release|Any CPU
-		{036A1269-EA1B-457A-8447-D90AEF80BAF9}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
@@ -597,19 +483,14 @@ Global
 	GlobalSection(NestedProjects) = preSolution
 		{86DC4DC7-DCEE-44AC-A7E3-B6D5A909422C} = {6BE49638-F842-4329-BE8A-30C0F30CCAA5}
 		{D6FFE1CC-98D6-425F-95D2-E40122D96E80} = {6BE49638-F842-4329-BE8A-30C0F30CCAA5}
-		{25ACF626-1C89-4A32-A434-E3DA17759631} = {C53DD924-C212-49EA-9BC4-1827421361EF}
 		{FF9DA4D9-3A84-441B-8AAE-B73B6D1EE8B3} = {BB99CE55-AD68-484A-9474-B8BACED247AB}
 		{697EB593-3277-4C43-B8F3-C578E21B051B} = {BB99CE55-AD68-484A-9474-B8BACED247AB}
 		{4FA6B420-53A3-4956-9C02-8649A720A9F5} = {BB99CE55-AD68-484A-9474-B8BACED247AB}
 		{AF298985-511F-476A-8F62-79908BE8DF01} = {A704E5B2-86A4-4D4F-902D-C10EA18C12DF}
 		{39B69A56-43E9-480D-9A60-8E0917795FE8} = {A704E5B2-86A4-4D4F-902D-C10EA18C12DF}
-		{F23ED302-7451-4D40-85C3-FACFCF884661} = {C53DD924-C212-49EA-9BC4-1827421361EF}
 		{5ED29E38-A65D-4123-BFED-799ADA885F08} = {86AEE6AF-16D7-47FD-8D3E-6B87B249208D}
-		{BA4250FD-9258-4469-A16D-48A0CAF5A2F8} = {C53DD924-C212-49EA-9BC4-1827421361EF}
 		{DC0D0EE5-1C0C-4D3E-BA0B-D6A59717DA94} = {6BE49638-F842-4329-BE8A-30C0F30CCAA5}
 		{3CCA947D-B466-4481-BC78-43CE98A8A3A7} = {80888CDA-37DC-4061-AF5F-7237BF16A320}
-		{86D901E6-C054-445B-92EC-E267EBB16278} = {80888CDA-37DC-4061-AF5F-7237BF16A320}
-		{4626A7D1-7AC0-4E21-9FED-083EF2A8194C} = {C53DD924-C212-49EA-9BC4-1827421361EF}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {32B9C883-432E-4FC8-A1BF-090EB033DD5B}
-- 
2.18.0

