From b7a7d59e94dd8d50adc09cf87624ded675748380 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 2 May 2019 01:41:05 -0500
Subject: [PATCH] Remove project that depends on Azure.

---
 Arcade.sln | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/Arcade.sln b/Arcade.sln
index 04082cdc..539d33eb 100644
--- a/Arcade.sln
+++ b/Arcade.sln
@@ -20,8 +20,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Deployment
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.XUnitExtensions", "src\Microsoft.DotNet.XUnitExtensions\src\Microsoft.DotNet.XUnitExtensions.csproj", "{F8E7FEC3-B9E0-4A08-8608-F48379204F3A}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.ServiceFabric.ServiceHost", "src\Microsoft.DotNet.ServiceFabric.ServiceHost\Microsoft.DotNet.ServiceFabric.ServiceHost.csproj", "{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Github.IssueLabeler", "src\Microsoft.DotNet.Github.IssueLabeler\Microsoft.DotNet.Github.IssueLabeler.csproj", "{A32C09E4-C334-4614-B40F-DEDF352C820E}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.GenFacades", "src\Microsoft.DotNet.GenFacades\Microsoft.DotNet.GenFacades.csproj", "{B6778A84-3330-4F2E-9D26-316C78A4BCC2}"
@@ -178,18 +176,6 @@ Global
 		{F8E7FEC3-B9E0-4A08-8608-F48379204F3A}.Release|x64.Build.0 = Release|Any CPU
 		{F8E7FEC3-B9E0-4A08-8608-F48379204F3A}.Release|x86.ActiveCfg = Release|Any CPU
 		{F8E7FEC3-B9E0-4A08-8608-F48379204F3A}.Release|x86.Build.0 = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|x64.Build.0 = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Debug|x86.Build.0 = Debug|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|Any CPU.Build.0 = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|x64.ActiveCfg = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|x64.Build.0 = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|x86.ActiveCfg = Release|Any CPU
-		{77371D91-1D9B-46E5-A1D5-E31CE3EEF524}.Release|x86.Build.0 = Release|Any CPU
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{A32C09E4-C334-4614-B40F-DEDF352C820E}.Debug|x64.ActiveCfg = Debug|Any CPU
-- 
2.18.0

