From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Thu, 24 Oct 2019 13:00:54 +0000
Subject: [PATCH 07/14] Import PackageVersions props if exists and update
 CodeAnalysis version

---
 eng/Versions.props | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 7728009d..0bf1c859 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -77,4 +77,8 @@
     <XliffTasksVersion>1.0.0-beta.20206.1</XliffTasksVersion>
     <MicrosoftSourceLinkVersion>1.1.0-beta-20206-02</MicrosoftSourceLinkVersion>
   </PropertyGroup>
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
+  <PropertyGroup>
+    <MsbuildTaskMicrosoftCodeAnalysisCSharpVersion Condition="'$(DotNetBuildFromSource)' == 'true'">$(MicrosoftCodeAnalysisCSharpVersion)</MsbuildTaskMicrosoftCodeAnalysisCSharpVersion>
+  </PropertyGroup>
 </Project>
