From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 9 Oct 2019 19:19:41 +0000
Subject: [PATCH 07/15] Update NuGetVersion to Reference Package version

---
 eng/Versions.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 2c8682d2..cc7149ca 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -37,8 +37,8 @@
     <McMasterExtensionsCommandLineUtils>2.3.0</McMasterExtensionsCommandLineUtils>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <MicrosoftBclJsonSourcesVersion>4.6.0-preview4.19202.2</MicrosoftBclJsonSourcesVersion>
-    <NuGetVersioningVersion>4.4.0</NuGetVersioningVersion>
-    <NuGetVersion>5.3.0</NuGetVersion>
+    <NuGetVersioningVersion>5.4.0-rtm</NuGetVersioningVersion>
+    <NuGetVersion>5.4.0-rtm</NuGetVersion>
     <OctokitVersion>0.32.0</OctokitVersion>
     <DotNetSleetLibVersion>2.2.139</DotNetSleetLibVersion>
     <SwashbuckleAspNetCoreSwaggerVersion>3.0.0</SwashbuckleAspNetCoreSwaggerVersion>
