From 47b4458cb15d7c9fb74a5c73cf52b9179033a799 Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Tue, 5 Sep 2017 17:52:05 -0400
Subject: [PATCH] Build pack project

---
 build/build.proj | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/build.proj b/build/build.proj
index fb1c221..32abffb 100644
--- a/build/build.proj
+++ b/build/build.proj
@@ -14,7 +14,7 @@
   </Target>
   
   <Target Name="Build">
-    <MSBuild Projects="@(SolutionFile)" Targets="Build" />
+    <MSBuild Projects="@(PackProject)" Targets="Build" />
   </Target>
 
   <Target Name="Test">
@@ -22,6 +22,6 @@
   </Target>
 
   <Target Name="Pack">
-    <MSBuild Projects="@(PackProject)" Targets="Pack" Properties="NoBuild=true" />
+    <MSBuild Projects="@(PackProject)" Targets="Pack" />
   </Target>
-</Project>
\ No newline at end of file
+</Project>
-- 
1.8.3.1

