From e4f1985229bbf528433461b5535c7baec55ca30e Mon Sep 17 00:00:00 2001
From: Nikola Milosavljevic <nikolam@microsoft.com>
Date: Mon, 26 Aug 2019 12:59:58 -0700
Subject: [PATCH] Remove unneeded feeds.

---
 eng/Versions.props | 2 --
 1 file changed, 2 deletions(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index a532291..8323eda 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -33,8 +33,6 @@
       https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json;
       https://myget.org/F/vs-devcore/api/v3/index.json;
       https://myget.org/F/vs-editor/api/v3/index.json;
-      https://vside.myget.org/F/vssdk/api/v3/index.json;
-      https://vside.myget.org/F/vs-impl/api/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
 </Project>
-- 
2.7.4

