From 3e8b288387f6f66a530efccca812adac5f9534c0 Mon Sep 17 00:00:00 2001
From: Nikola Milosavljevic <nikolam@microsoft.com>
Date: Mon, 26 Aug 2019 12:47:31 -0700
Subject: [PATCH] Remove vssdk feed

---
 eng/Versions.props | 1 -
 1 file changed, 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index a532291..5adb4ff 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -33,7 +33,6 @@
       https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json;
       https://myget.org/F/vs-devcore/api/v3/index.json;
       https://myget.org/F/vs-editor/api/v3/index.json;
-      https://vside.myget.org/F/vssdk/api/v3/index.json;
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
-- 
2.7.4

