From 96778bca79e66c3b05f17df3eed684bc9298dbbd Mon Sep 17 00:00:00 2001
From: Tom Deseyn <tom.deseyn@gmail.com>
Date: Mon, 19 Oct 2020 15:16:45 +0200
Subject: [PATCH] nuget.client: don't use PublicApiAnalyzers

---
 build/common.targets | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/common.targets b/build/common.targets
index 197878d61..25369b444 100644
--- a/build/common.targets
+++ b/build/common.targets
@@ -54,6 +54,11 @@
       <Import Project="$(NuGetBuildTasksPackTargets)" />
   </ImportGroup>
 
+  <!-- Don't use PublicApiAnalyzer on source-build .NET -->
+  <PropertyGroup Condition=" '$(IsBuildOnlyXPLATProjects)' == 'true' ">
+    <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
+  </PropertyGroup>
+
   <!-- Projects we pack become public APIs that others can use -->
   <PropertyGroup Condition=" '$(IsBuildOnlyXPLATProjects)' != 'true' and '$(PackProject)' == 'true' ">
     <UsePublicApiAnalyzer Condition=" '$(UsePublicApiAnalyzer)' == '' " >true</UsePublicApiAnalyzer>
-- 
2.26.2

