From 744a402001e49a39210e9f5e5f4cf1a1559f2988 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Sat, 10 Jun 2017 22:37:59 -0700
Subject: [PATCH] Bump MSBuild Reference

---
 build/config.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config.props b/build/config.props
index b46be34..71a1e04 100644
--- a/build/config.props
+++ b/build/config.props
@@ -18,7 +18,7 @@
     <TestSDKVersion>15.0.0</TestSDKVersion>
     <MoqVersion>4.7.63</MoqVersion>
     <FluentAssertionsVersion>4.19.2</FluentAssertionsVersion>
-    <MicrosoftBuildPackageVersion>0.1.0-preview-00038-160914</MicrosoftBuildPackageVersion>
+    <MicrosoftBuildPackageVersion>15.1.1012</MicrosoftBuildPackageVersion>
   </PropertyGroup>
 
   <!-- Config -->
-- 
1.8.3.1

