From ce18ce78f27717cb1079270d6a58d5c36e70425a Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Thu, 10 Dec 2020 16:17:55 -0600
Subject: [PATCH] Use source-built Microsoft.Web.Xdt

---
 build/packages.targets | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/build/packages.targets b/build/packages.targets
index 19df3657c..7d8c01f84 100644
--- a/build/packages.targets
+++ b/build/packages.targets
@@ -9,6 +9,10 @@
         <VSThreadingVersion>16.7.56</VSThreadingVersion>
     </PropertyGroup>
 
+    <PropertyGroup>
+        <MicrosoftWebXdtPackageVersion Condition="'$(MicrosoftWebXdtPackageVersion)' == ''">3.0.0</MicrosoftWebXdtPackageVersion>
+    </PropertyGroup>
+
     <!-- Test and package versions -->
     <PropertyGroup>
         <XunitVersion>2.4.1</XunitVersion>
@@ -58,7 +62,7 @@
         <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers" Version="$(VSThreadingVersion)" />
         <PackageReference Update="Microsoft.VisualStudio.Workspace.VSIntegration" Version="16.6.55-preview-0003" />
         <PackageReference Update="Microsoft.VisualStudio.Utilities" Version="$(VSFrameworkVersion)" />
-        <PackageReference Update="Microsoft.Web.Xdt" Version="2.1.2" />
+        <PackageReference Update="Microsoft.Web.Xdt" Version="$(MicrosoftWebXdtPackageVersion)" />
         <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
         <PackageReference Update="System.Collections.Immutable" Version="1.7.1" />
         <PackageReference Update="System.Diagnostics.Debug" Version="$(SystemPackagesVersion)" />
-- 
2.25.4

