From e9489f4725e6ba261818b074836ada1f6656a946 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 15 May 2020 17:16:40 +0000
Subject: [PATCH] Don't add sources when building offline.

---
 eng/Versions.props | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 9c23b0c..f569224 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -23,5 +23,9 @@
     <MicrosoftDotNetApiCompatVersion>5.0.0-beta.20201.2</MicrosoftDotNetApiCompatVersion>
     <!-- We depend on (and redistribute) the official Mono.Cecil NuGet package built from https://github.com/jbevain/cecil -->
     <MonoCecilVersion>0.11.2</MonoCecilVersion>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      $(RestoreSources);
+      https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json
+    </RestoreSources>
   </PropertyGroup>
 </Project>
-- 
1.8.3.1

