From 7847cbd68ff975062dcf4d7cced1038b741511a5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 3 Feb 2019 13:33:44 -0600
Subject: [PATCH] Add PackageProjectUrl to avoid error.

---
 Directory.Build.props | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 2627e8a28..b227447c5 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -30,6 +30,8 @@
     <!-- NU5104: Disable 'A stable release of a package should not have a prerelease dependency' warning globally -->
     <!-- The "pack" command under 'buildCrossTargeting' for 'Microsoft.DotNet.MSBuildSdkResolver' throws a "NU5104" warning/error while using a "2.1.300" stage0 SDK. For: Linux-x64, Linux-arm, and Linux-arm64 -->
     <NoWarn>NU1701;NU5104;NU5125;NU5105</NoWarn>
+
+    <PackageProjectUrl>https://github.com/dotnet/toolset</PackageProjectUrl>
   </PropertyGroup>
 
 </Project>
-- 
2.18.0

