From ff11e673b834e2abc7b2273d912c26d8ccd2c33d Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 19 Nov 2019 22:06:41 +0000
Subject: [PATCH] Exclude test folder from SourceBuild

---
 test/Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/Directory.Build.props b/test/Directory.Build.props
index 406c4f8..b8995f9 100644
--- a/test/Directory.Build.props
+++ b/test/Directory.Build.props
@@ -3,5 +3,6 @@
 
   <PropertyGroup>
     <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 </Project>
-- 
1.8.3.1

