From 51d590ab755edd0e8d3420ee832e3dfc37c0a291 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 23 May 2019 20:20:56 -0500
Subject: [PATCH] Exclude test projects from source-build.

---
 test/Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/Directory.Build.props b/test/Directory.Build.props
index 406c4f8c1..b8995f951 100644
--- a/test/Directory.Build.props
+++ b/test/Directory.Build.props
@@ -3,5 +3,6 @@
 
   <PropertyGroup>
     <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 </Project>
-- 
2.18.0

