From a6cf4ca264c38277969e1c997bc5c1c180246664 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 8 Aug 2018 14:32:03 -0500
Subject: [PATCH] Experimental: re-enable RID workaround.

---
 build/DotnetCoreSdkLKG.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DotnetCoreSdkLKG.props b/build/DotnetCoreSdkLKG.props
index f4b8e89d3..7f8ad6927 100644
--- a/build/DotnetCoreSdkLKG.props
+++ b/build/DotnetCoreSdkLKG.props
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CoreSetupRid>$(HostRid)</CoreSetupRid>
-    <CoreSetupRid Condition=" '$(HostOSName)' == 'win' or '$(HostOSName)' == 'osx' ">$(HostMonikerRid)</CoreSetupRid>
+    <CoreSetupRid Condition=" ('$(HostOSName)' == 'win' or '$(HostOSName)' == 'osx') and '$(DotNetBuildFromSource)' != 'true' ">$(HostMonikerRid)</CoreSetupRid>
 
     <SharedFrameworkRid>$(CoreSetupRid)</SharedFrameworkRid>
     <SharedFrameworkRid Condition=" '$(Rid)' == 'linux-musl-x64' ">$(Rid)</SharedFrameworkRid>
-- 
2.18.0

