From 7eed6efb0af81e4637749865c69a67861023530f Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 19 Nov 2019 22:06:41 +0000
Subject: [PATCH 1/4] Exclude test folder from SourceBuild

---
 test/Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/Directory.Build.props b/test/Directory.Build.props
index 406c4f8c1..b8995f951 100644
--- a/test/Directory.Build.props
+++ b/test/Directory.Build.props
@@ -3,5 +3,6 @@
 
   <PropertyGroup>
     <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 </Project>
-- 
2.21.0

