From cb0af530bbd4c9c552494d0e45778a08060c9c3a Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 20 Oct 2019 23:00:39 -0500
Subject: [PATCH 2/2] Pick up netcoreapp3.0 vstest files

---
 src/redist/targets/GenerateLayout.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/redist/targets/GenerateLayout.targets b/src/redist/targets/GenerateLayout.targets
index 376bb9eeb..23b9841f5 100644
--- a/src/redist/targets/GenerateLayout.targets
+++ b/src/redist/targets/GenerateLayout.targets
@@ -35,7 +35,7 @@
       <TestCliBitsToExclude Include="$(TestCliNuGetDirectory)System.Memory.dll" />
       <TestCliBitsToExclude Include="$(TestCliNuGetDirectory)System.Runtime.CompilerServices.Unsafe.dll" />
       <TestCliBitsToExclude Include="$(TestCliNuGetDirectory)NuGet.Frameworks.dll" />
-      <TestCliBits Include="$(TestCliNuGetDirectory)**/*"
+      <TestCliBits Include="$(NuGetPackagesDir)/microsoft.testplatform.cli/$(MicrosoftTestPlatformCLIPackageVersion)/contentFiles/any/**/*"
                    Exclude="@(TestCliBitsToExclude)" />
     </ItemGroup>
     <Copy SourceFiles="@(TestCliBits)" DestinationFiles="@(TestCliBits->'$(OutputPath)/%(RecursiveDir)%(Filename)%(Extension)')" />
-- 
2.18.0

