From 85b1dfa1a0068f9db348b2d59ecb836e6e2a52f7 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 17 Feb 2019 08:30:37 -0600
Subject: [PATCH] Add PVP import.

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 44e049c9b..1d62eaf73 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -40,6 +40,9 @@
     <SystemRuntimeInteropServicesPackageVersion>4.3.0</SystemRuntimeInteropServicesPackageVersion>
     <XliffTasksPackageVersion>0.2.0-beta-000042</XliffTasksPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
-- 
2.18.0

