From 4ef49ec168072ed685ecf9c6a9db1075b34bf4b1 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 11 Feb 2019 12:56:51 -0600
Subject: [PATCH 2/5] Consolidate Newtonsoft.Json versions.

---
 src/dotnet/dotnet.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dotnet/dotnet.csproj b/src/dotnet/dotnet.csproj
index 058396c39..cd1e68ef8 100644
--- a/src/dotnet/dotnet.csproj
+++ b/src/dotnet/dotnet.csproj
@@ -74,4 +74,4 @@
   <ItemGroup Condition=" '$(IncludeAspNetCoreRuntime)' != 'false' ">
     <PackageReference Include="Microsoft.AspNetCore.DeveloperCertificates.XPlat" Version="$(MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion)" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
2.21.0

