From aba730c0c172afe5324315ae99e5ad7c7f771c9b Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 24 May 2020 17:11:35 +0000
Subject: [PATCH] Remove feed that no longer exists.

---
 build/NugetConfigFile.targets | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/NugetConfigFile.targets b/build/NugetConfigFile.targets
index 0cae8455a..535665c79 100644
--- a/build/NugetConfigFile.targets
+++ b/build/NugetConfigFile.targets
@@ -30,7 +30,6 @@
     <add key="linux-musl-bootstrap-feed" value="https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180420-03/aspnet-inputs/index.json" />
     <add key="dotnet-msbuild" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
     <add key="myget-vstest" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
-    <add key="BlobFeed-2.1.3-runtime" value="https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180725-02/final/index.json"/>
         ]]>
       </NugetConfigCLIFeeds>
 
-- 
2.18.2

