From ecc334fe9d6ef2b32bc0692f1968341eb0c06207 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 11 Feb 2019 12:56:51 -0600
Subject: [PATCH] Consolidate Newtonsoft.Json versions.

---
 src/dotnet/dotnet.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dotnet/dotnet.csproj b/src/dotnet/dotnet.csproj
index 2a247fc25..1fd84ec14 100644
--- a/src/dotnet/dotnet.csproj
+++ b/src/dotnet/dotnet.csproj
@@ -75,4 +75,4 @@
   <ItemGroup Condition=" '$(IncludeAspNetCoreRuntime)' != 'false' ">
     <PackageReference Include="Microsoft.AspNetCore.DeveloperCertificates.XPlat" Version="$(MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion)" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

