From 9835c11e7d5b115a3b022d4c10ef0fd92bb01089 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 1 May 2019 07:12:01 -0500
Subject: [PATCH 4/5] Add LangVersion to CLI.

---
 Directory.Build.props | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 1d1252966..789f9f12b 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -37,6 +37,8 @@ tools\TestAssetsDependencies\TestAssetsDependencies.csproj
     <TestCaptureOutput>false</TestCaptureOutput>
 
     <IsShippingPackage>false</IsShippingPackage>
+
+    <LangVersion>Latest</LangVersion>
   </PropertyGroup>
   
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
-- 
2.21.0

