From ce0fc174e8a6cf849dd05fde65ddd4d08d12a1d1 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Wed, 24 May 2017 17:28:07 -0700
Subject: [PATCH 4/5] Provide a way to explicitly set commit count and sha

---
 build/GitCommitInfo.targets | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/build/GitCommitInfo.targets b/build/GitCommitInfo.targets
index 4addb6c..437a657 100644
--- a/build/GitCommitInfo.targets
+++ b/build/GitCommitInfo.targets
@@ -1,12 +1,14 @@
 <Project ToolsVersion="15.0">
   <Target Name="WriteGitCommitInfoProps">
     <Exec Command="git rev-list --count HEAD" 
-          ConsoleToMSBuild="true">
+          ConsoleToMSBuild="true"
+          Condition="'$(GitInfoCommitCount)' == ''">
       <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitCount" />
     </Exec>
 
     <Exec Command="git rev-parse HEAD" 
-          ConsoleToMSBuild="true">
+          ConsoleToMSBuild="true"
+          Condition="'$(GitInfoCommitHash)' == ''">
       <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitHash" />
     </Exec>
 
-- 
1.8.3.1

