From 860656146b1d7d882daf3ab6be3607604ab2d4e7 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 11 Feb 2019 12:56:51 -0600
Subject: [PATCH 4/7] Consolidate Newtonsoft.Json versions.

---
 src/dotnet/dotnet.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dotnet/dotnet.csproj b/src/dotnet/dotnet.csproj
index 3319cdcfc..47db7d85e 100644
--- a/src/dotnet/dotnet.csproj
+++ b/src/dotnet/dotnet.csproj
@@ -76,4 +76,4 @@
   <ItemGroup Condition=" '$(IncludeAspNetCoreRuntime)' != 'false' ">
     <PackageReference Include="Microsoft.AspNetCore.DeveloperCertificates.XPlat" Version="$(MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion)" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

