From 9ed5ec902e870b78969d931c4cc140c8f0e24417 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 23 Jul 2019 12:57:21 -0500
Subject: [PATCH 5/5] Remove unneeded project.

---
 test/ArgumentsReflector/ArgumentsReflector.csproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/ArgumentsReflector/ArgumentsReflector.csproj b/test/ArgumentsReflector/ArgumentsReflector.csproj
index f11319279..509432d04 100644
--- a/test/ArgumentsReflector/ArgumentsReflector.csproj
+++ b/test/ArgumentsReflector/ArgumentsReflector.csproj
@@ -4,6 +4,7 @@
     <TargetFramework>$(CliTargetFramework)</TargetFramework>
     <AssemblyName>ArgumentsReflector</AssemblyName>
     <OutputType>Exe</OutputType>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.21.0

