From 232473198cabd8fe9f79027932d49ba5bd695426 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 8 Jun 2017 22:21:04 -0700
Subject: [PATCH] WIP: Add some NuGet projects to redist

---
 src/redist/redist.csproj | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
index c45cd6b..3e887a3 100644
--- a/src/redist/redist.csproj
+++ b/src/redist/redist.csproj
@@ -22,6 +22,10 @@
     <PackageReference Include="Microsoft.TestPlatform.CLI" Version="$(CLI_TestPlatform_Version)" />
     <PackageReference Include="Microsoft.TestPlatform.Build" Version="$(CLI_TestPlatform_Version)" />
     <PackageReference Include="NuGet.ProjectModel" Version="$(CLI_NuGet_Version)" />
+    <PackageReference Include="NuGet.Commands" Version="$(CLI_NuGet_Version)" />
+    <PackageReference Include="NuGet.DependencyResolver.Core" Version="$(CLI_NuGet_Version)" />
+    <PackageReference Include="NuGet.LibraryModel" Version="$(CLI_NuGet_Version)" />
+    <PackageReference Include="NuGet.Protocol" Version="$(CLI_NuGet_Version)" />
     <!-- The project json migration commands depend on an older version of Roslyn.
          Lift the version here to match what tool_roslyn depends on (otherwise an older version will
          be added to the TPA when we crossgen and we won't be able to crossgen tool_roslyn -->
-- 
1.8.3.1

