From 4eec9f8272e9b692d5fe19e46aa94d490e1c4f38 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Tue, 20 Jun 2017 15:52:16 -0700
Subject: [PATCH] Unify templating versions

---
 build/DependencyVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index a05b626..994eefa 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -20,7 +20,7 @@
     <SharedFrameworkVersion>$(CLI_SharedFrameworkVersion)</SharedFrameworkVersion>
     <SharedHostVersion>$(CLI_SharedFrameworkVersion)</SharedHostVersion>
     <HostFxrVersion>$(CLI_SharedFrameworkVersion)</HostFxrVersion>
-    <TemplateEngineVersion>1.0.0-beta2-20170614-260</TemplateEngineVersion>
+    <TemplateEngineVersion>1.0.0-beta2-20170619-264</TemplateEngineVersion>
     <TemplateEngineTemplateVersion>1.0.0-beta2-20170619-264</TemplateEngineTemplateVersion>
     <TemplateEngineTemplate2_0Version>1.0.0-beta2-20170619-264</TemplateEngineTemplate2_0Version>
     <PlatformAbstractionsVersion>2.0.0-preview2-25407-01</PlatformAbstractionsVersion>
-- 
1.8.3.1

