From d8d39ebf8c98a5339089cb89d5838e4ece71780f Mon Sep 17 00:00:00 2001
From: karajas <karajas@microsoft.com>
Date: Wed, 6 Sep 2017 18:33:13 -0400
Subject: [PATCH] Allow cli to restore to an intermdiate package cache to pick up source built packages 

---
 build/Prepare.targets | 22 +++++++++++++++++++++-
 1 file changed, 21 insertions(+), 1 deletion(-)

diff --git a/build/Prepare.targets b/build/Prepare.targets
index 214b1f7..b94b60a 100644
--- a/build/Prepare.targets
+++ b/build/Prepare.targets
@@ -83,6 +83,26 @@
                    ProjectPath="&quot;%(RestoreToolsPackagesInput.FullPath)&quot;"
                    AdditionalParameters="/p:UsePortableLinuxSharedFramework=$(UsePortableLinuxSharedFramework)" />
 
+
+      <PropertyGroup>
+        <IntermediateToolsPackages>$(MSBuildThisFileDirectory)packages</IntermediateToolsPackages>
+        <FilesToCopyPattern>$(MSBuildThisFileDirectory)packages/**/*</FilesToCopyPattern>
+      </PropertyGroup>
+  
+      <DotNetRestore ToolPath="$(Stage0Directory)"
+                     ProjectPath="&quot;$(RepoRoot)/tools/CrossGen.Dependencies/CrossGen.Dependencies.csproj&quot;"
+                     Source="$(SourceBuiltPackages)"
+                     Runtime="$(SharedFrameworkRid)"
+                     Packages="$(IntermediateToolsPackages)"
+                     AdditionalParameters="/p:UsePortableLinuxSharedFramework=$(UsePortableLinuxSharedFramework)" />
+                     
+      <ItemGroup>
+        <FilesToCopy Include="$(FilesToCopyPattern)" />
+      </ItemGroup>
+  
+    <Message importance="high" text="KARTHIKDEBUG '@(FilesToCopy)' '$(NuGetPackagesDir)'" />
+    <Copy SourceFiles="@(FilesToCopy)" OverwriteReadOnlyFiles="true" DestinationFolder="$(NuGetPackagesDir)/%(RecursiveDir)"/>
+
   </Target>
 
   <Target Name="CleanToolsLockFiles" >
@@ -94,7 +114,7 @@
 
   <Target Name="SetupRestoreToolsPackagesInputsOutputs">
     <ItemGroup>
-      <RestoreToolsPackagesInput Include="$(RepoRoot)/tools/**/*.csproj" />
+      <RestoreToolsPackagesInput Include="$(RepoRoot)/tools/**/*.csproj" Exclude="$(RepoRoot)/tools/CrossGen.Dependencies/CrossGen.Dependencies.csproj" />
     </ItemGroup>
   </Target>
 
-- 
1.8.3.1

