From ff5dbadc760046a2557194bae052de8a82d866be Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Thu, 26 Apr 2018 15:45:05 +0000
Subject: [PATCH] Exclude Localization in source-build

---
 src/redist/redist.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
index da07741..c2b2a87 100644
--- a/src/redist/redist.csproj
+++ b/src/redist/redist.csproj
@@ -19,7 +19,7 @@
     <PackageReference Include="NuGet.Build.Tasks" Version="$(NuGetBuildTasksPackageVersion)" />
     <PackageReference Include="Microsoft.TestPlatform.CLI" Version="$(MicrosoftTestPlatformCLIPackageVersion)" />
     <PackageReference Include="Microsoft.TestPlatform.Build" Version="$(MicrosoftTestPlatformBuildPackageVersion)" />
-    <PackageReference Include="NuGet.Localization" Version="$(NuGetProjectModelPackageVersion)" />
+    <PackageReference Condition=" '$(DotNetBuildFromSource)' != 'true' " Include="NuGet.Localization" Version="$(NuGetProjectModelPackageVersion)" />
     <PackageReference Include="NuGet.ProjectModel" Version="$(NuGetProjectModelPackageVersion)" />
     <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(MicrosoftNETCoreCompilersPackageVersion)">
       <ExcludeAssets>All</ExcludeAssets>
-- 
1.8.3.1

