From ba3e610262c4ab9a4872fc549ad0284fca91985e Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Mon, 5 Aug 2019 18:21:28 +0000
Subject: [PATCH 2/8] restore project for sourcebuild&&IncludeTestUtils

---
 build.proj | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build.proj b/build.proj
index 09b69e0..e15d7ce 100644
--- a/build.proj
+++ b/build.proj
@@ -22,7 +22,8 @@
       <_RestoreProjects Include="external\dir.proj" Build="true" />
     </ItemGroup>
     
-    <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
+    <!--When Building from source and including test utils then ref and src projects are to be restored-->
+    <ItemGroup Condition="('$(DotNetBuildFromSource)' != 'true') or '$(IncludeTestUtils)' == 'true'">
       <_RestoreProjects Include="$(SourceDir)tests.proj" Build="false" />
     </ItemGroup>
 
-- 
1.8.3.1

