From b99f616f94a162a1339ad3b6671db435843fa9bb Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 10 Jul 2019 16:45:18 -0500
Subject: [PATCH 3/3] Disable TargetFramework for packaging - Arcade change.

---
 eng/configurations/targetgroups.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/configurations/targetgroups.props b/eng/configurations/targetgroups.props
index 5878d40d25..24b8b79d7a 100644
--- a/eng/configurations/targetgroups.props
+++ b/eng/configurations/targetgroups.props
@@ -248,6 +248,7 @@
     <TargetGroups Include="package">
       <!-- this targetgroup is not considered compatible with any other targetgroup so it should only
            build when specified directly or BuildAllConfigurations is set to True.  -->
+      <TargetFramework>none</TargetFramework>
     </TargetGroups>
     <TargetGroups Include="mono">
       <TargetFramework>net463</TargetFramework>
-- 
2.18.0

