From ea4b72ad16e2dee504e9fd02a19d102a100d4bd0 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Tue, 6 Aug 2019 19:21:43 +0000
Subject: [PATCH 3/8] Include pretest build for sourcebuild corefx tests

---
 src/dirs.proj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dirs.proj b/src/dirs.proj
index af8d308..30932b8 100644
--- a/src/dirs.proj
+++ b/src/dirs.proj
@@ -13,7 +13,7 @@
     <Project Include="src.builds" />
     <Project Include="shims\manual\*.csproj" />
     <Project Include="shims\ApiCompat.proj" Condition="'$(DotNetBuildFromSource)' != 'true'" />
-    <Project Include="pretest.proj" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(BuildAllConfigurations)' != 'true'" />
+    <Project Include="pretest.proj" Condition="('$(DotNetBuildFromSource)' != 'true' and '$(BuildAllConfigurations)' != 'true') or '$(IncludeTestUtils)' == 'true'" />
   </ItemGroup>
 
   <!-- Don't build System.Text.Encodings.Web tools as part of build; they're intended to be run manually. -->
-- 
1.8.3.1

