From da25df507ba424fb8c3f04a26a761f3de996d08d Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 20 Jun 2019 07:26:25 -0500
Subject: [PATCH 3/3] Exclude XUnit project from source-build.

---
 external/test-runtime/XUnit.Runtime.depproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/test-runtime/XUnit.Runtime.depproj b/external/test-runtime/XUnit.Runtime.depproj
index fb9c74a..8a6d08e 100644
--- a/external/test-runtime/XUnit.Runtime.depproj
+++ b/external/test-runtime/XUnit.Runtime.depproj
@@ -27,6 +27,7 @@
     <Language Condition="'$(Language)' == ''">c#</Language>
     <!-- Don't warn if some dependencies were rolled forward -->
     <NoWarn>$(NoWarn);NU1603;NU1701</NoWarn>
+    <ExcludeFromSourceBuild Condition="'$(IncludeTestUtils)' != 'true'">true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.7.4

