From 90cd66c9b8f87a1024f2309871169606fc1f8a6b Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Wed, 7 Jun 2017 11:35:28 -0700
Subject: [PATCH] Prevent some packages from being restored

---
 external/dir.proj | 2 --
 1 file changed, 2 deletions(-)

diff --git a/external/dir.proj b/external/dir.proj
index d3d7fd8769..3c9d28f1ba 100644
--- a/external/dir.proj
+++ b/external/dir.proj
@@ -10,11 +10,9 @@
     <Project Include="netstandard/netstandard.depproj" />
     <Project Include="netfx/netfx.depproj" />
     <Project Include="runtime/runtime.depproj" />
-    <Project Include="test-runtime/XUnit.Runtime.depproj" />
     <Project Include="harvestPackages/harvestPackages.depproj" />
     <Project Include="portable\portable.depproj" />
     <Project Include="ilasm/ilasm.depproj" />
-    <Project Include="docs/docs.depproj" />
     <Project Condition="'$(ILLinkTrimAssembly)' != 'false'" Include="ILLink/ILLink.depproj" />
   </ItemGroup>
 
-- 
2.14.1.windows.1

