From 72c9c7451858aeb7501fb439be6fc8f89cee5774 Mon Sep 17 00:00:00 2001
From: "Nikola Milosavljevic (CLR)" <nikolam@microsoft.com>
Date: Mon, 1 Jul 2019 16:27:56 -0700
Subject: [PATCH 4/4] Fix _runtimeOSFamily property

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 111ff612e6..ea1bb21b70 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -179,6 +179,7 @@
   <PropertyGroup>
     <_runtimeOSVersionIndex>$(RuntimeOS.IndexOfAny(".-0123456789"))</_runtimeOSVersionIndex>
     <_runtimeOSFamily Condition="'$(_runtimeOSVersionIndex)' != '-1'">$(RuntimeOS.SubString(0, $(_runtimeOSVersionIndex)))</_runtimeOSFamily>
+    <_runtimeOSFamily Condition="'$(_runtimeOSFamily)' == ''">$(RuntimeOS)</_runtimeOSFamily>
     <_portableOS>linux</_portableOS>
     <_portableOS Condition="'$(RuntimeOS)' == 'linux-musl'">linux-musl</_portableOS>
     <_portableOS Condition="'$(_runtimeOSFamily)' == 'win' OR '$(OSGroup)' == 'Windows_NT'">win</_portableOS>
-- 
2.20.1.vfs.1.1.104.g2ab7360

