From d33f4c2f32468e54f41a616a00fe272ac240f32b Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 19 Jul 2019 04:17:40 -0500
Subject: [PATCH 4/5] Another nullability warning.

---
 Directory.Build.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 515c0806f9..c7761d4926 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -4,7 +4,7 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <RepositoryEngineeringDir>$(RepoRoot)eng\</RepositoryEngineeringDir>
     <RepositoryUrl>git://github.com/dotnet/corefx</RepositoryUrl>
-    <NoWarn>$(NoWarn),CS8597</NoWarn>
+    <NoWarn>$(NoWarn),CS8597,CS8632</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>
@@ -266,7 +266,7 @@
     <Features>strict</Features>
     <Features Condition="'$(IsTestProject)' != 'true'">$(Features);nullablePublicOnly</Features>
     <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <Deterministic>true</Deterministic>
 
     <!-- We decided to keep this disabled by default to see some history of way have a look at https://github.com/dotnet/corefx/issues/3140 -->
-- 
2.21.0

