From 3a15efa8338a8dd8933f791e83fa222d4938a13f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 14 Aug 2019 12:46:37 -0500
Subject: [PATCH 4/4] Move default AssetTargetFallback here.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index e0acf988bb..a373f43919 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -4,6 +4,7 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <RepositoryEngineeringDir>$(RepoRoot)eng\</RepositoryEngineeringDir>
     <RepositoryUrl>git://github.com/dotnet/corefx</RepositoryUrl>
+    <AssetTargetFallback>netstandard2.0</AssetTargetFallback>
   </PropertyGroup>
 
   <PropertyGroup>
-- 
2.21.0

