From 31bbc9908f45e799cb10161c5b3c2ec302121e98 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sat, 17 Nov 2018 16:34:00 -0600
Subject: [PATCH] wip

---
 eng/common/tools.sh | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index c7d03ee028..9aaae295bc 100644
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -240,14 +240,6 @@ if [[ -z $HOME ]]; then
   mkdir -p "$HOME"
 fi
 
-if [[ -z ${NUGET_PACKAGES:-} ]]; then
-  if [[ $ci == true ]]; then
-    export NUGET_PACKAGES="$repo_root/.packages"
-  else
-    export NUGET_PACKAGES="$HOME/.nuget/packages"
-  fi
-fi
-
 mkdir -p "$toolset_dir"
 mkdir -p "$log_dir"
 
-- 
2.17.1.windows.2

