From ffbd1e02eda3abf45ca1f33bcb5ae41599a81ebd Mon Sep 17 00:00:00 2001
From: Tomas Weinfurt <tweinfurt@yahoo.com>
Date: Tue, 12 Nov 2019 13:24:37 -0800
Subject: [PATCH 5/6] do not fail on test chmod

---
 external/runtime/runtime.depproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/external/runtime/runtime.depproj b/external/runtime/runtime.depproj
index 793eb4e6f3..5a7426e5f9 100644
--- a/external/runtime/runtime.depproj
+++ b/external/runtime/runtime.depproj
@@ -62,7 +62,7 @@
           SkipUnchangedFiles="true"
           UseHardlinksIfPossible="$(UseHardlink)" />
 
-    <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension)" Condition="'$(OS)' != 'Windows_NT'"/>
+    <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension) || true" Condition="'$(OS)' != 'Windows_NT'"/>
 
     <Copy SourceFiles="@(HostSdkFile)"
           DestinationFolder="$(TestHostRootPath)sdk\$(DotNetVersion)\%(RecursiveDir)"
-- 
2.21.0

