From eb84487f1a4b7342c6cc0ae25077efcf6c4f829c Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Mon, 12 Aug 2019 19:45:29 +0000
Subject: [PATCH 6/8] Enable inclusion of IncluteTestUtils Flag for coreFx

---
 Directory.Build.props | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index e0acf98..dd2fccb 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -346,6 +346,15 @@
     <PackageConflictPlatformManifests Include="$(PlatformManifestFile)" />
   </ItemGroup>
 
+  <!-- 
+    Controls inclusion/exclusion of CoreFx TestUtilities project.
+    Allows overriding from the calling process, i.e. source-build test run.
+  -->
+  <PropertyGroup>
+    <IncludeTestUtils Condition="('$(IncludeTestUtils)' == '' and '$(DotNetBuildFromSource)' != 'true') or '$(BuildTests)' == 'true'">true</IncludeTestUtils>
+    <IncludeTestUtils Condition="'$(IncludeTestUtils)' == ''">false</IncludeTestUtils>
+  </PropertyGroup>
+
   <!-- AOT only asset that we currently don't support. -->
   <ItemGroup>
     <ProjectExclusions Include="$(SourceDir)System.Private.Reflection.Metadata.Ecma335/**/*.csproj" />
-- 
1.8.3.1

