From 6da83166203ce5c534b4a6079adc9bd8e154b95d Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 6 Dec 2019 16:17:24 -0600
Subject: [PATCH 6/6] Manually stabilize MS.NC.Platforms

---
 .../Microsoft.NETCore.Platforms.pkgproj                         | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj b/pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj
index 14f8501026..c0d04f03fc 100644
--- a/pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj
+++ b/pkg/Microsoft.NETCore.Platforms/Microsoft.NETCore.Platforms.pkgproj
@@ -1,7 +1,7 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>$(PlatformPackageVersion)</PackageVersion>
+    <PackageVersion>3.0.1</PackageVersion>
     <SkipValidatePackage>true</SkipValidatePackage>
     <!-- We don't need to harvest the stable packages to build this -->
     <HarvestStablePackage>false</HarvestStablePackage>
-- 
2.21.0

