From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Logan Bussell <36081148+lbussell@users.noreply.github.com>
Date: Tue, 16 Nov 2021 20:06:38 +0000
Subject: [PATCH 9/9] reference 1.0.0 version of sourcelink

---
 eng/Versions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index a6bd6a89e3..4dc1db0b25 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -45,7 +45,7 @@
     <MicrosoftDotNetBuildTasksFeedVersion>2.2.0-beta.21552.6</MicrosoftDotNetBuildTasksFeedVersion>
     <MicrosoftDotNetVersionToolsTasksPackageVersion>1.0.0-beta.21552.6</MicrosoftDotNetVersionToolsTasksPackageVersion>
     <!-- sourcelink -->
-    <MicrosoftSourceLinkVSTSVersion>1.0.0-beta2-19519-01</MicrosoftSourceLinkVSTSVersion>
+    <MicrosoftSourceLinkVSTSVersion>1.0.0</MicrosoftSourceLinkVSTSVersion>
     <!-- roslyn -->
     <MicrosoftNetCompilersToolsetVersion>3.3.0-beta2-19367-02</MicrosoftNetCompilersToolsetVersion>
     <!-- Core-setup dependencies -->
-- 
2.31.1

