From c845bf0373ac8d61c509fd83fca87d69f3caefe3 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 11 Apr 2018 21:51:39 +0000
Subject: [PATCH] Update build script to not kill on source-build

---
 build/build.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/build.sh b/build/build.sh
index 0e63f37..9a1473f 100755
--- a/build/build.sh
+++ b/build/build.sh
@@ -137,7 +137,7 @@ function ExitIfError {
   then
     echo "$2"
 
-    if ! $ci # kill command not permitted on CI machines
+    if [[ "$ci" != "true" && "$dotnetBuildFromSource" != "true" ]]; # kill command not permitted on CI machines or in source-build
     then
       StopProcesses
     fi
@@ -269,7 +269,7 @@ function InstallRepoToolset {
   if [ ! -d "$RepoToolsetBuildProj" ]
   then
     ToolsetProj="$ScriptRoot/Toolset.proj"
-    CallMSBuild $(QQ $ToolsetProj) /t:restore /m /nologo /clp:Summary /warnaserror /v:$verbosity $logCmd
+    CallMSBuild $(QQ $ToolsetProj) /t:restore /m /nologo /clp:Summary /warnaserror /v:$verbosity $logCmd $properties
   fi
 }
 
@@ -438,4 +438,4 @@ fi
 
 AssertNugetPackages
 
-ExitIfError $? "AssertNugetPackages failed"
\ No newline at end of file
+ExitIfError $? "AssertNugetPackages failed"
-- 
1.8.3.1

