From 860f5df19ceb8734fa32ea8344b4ded6976b5a7b Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 22 May 2019 19:25:49 -0500
Subject: [PATCH] Add PackageLicenseExpression.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 5275e5d2..f37c06c0 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -37,6 +37,7 @@
     <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</PackageProjectUrl>
     <PackageIconUrl>https://go.microsoft.com/fwlink/?linkid=825694</PackageIconUrl>
     <PackageTags>MSBuild</PackageTags>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
 
     <TargetMSBuildToolsVersion>Current</TargetMSBuildToolsVersion>
   </PropertyGroup>
-- 
2.18.0

