From 9a4def58a44d14a38ff8d5cc83ed5cfa7b0b863f Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 11 Apr 2018 22:28:02 +0000
Subject: [PATCH] source-build restore sources for Toolset proj

---
 build/Toolset.proj | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build/Toolset.proj b/build/Toolset.proj
index e8395c3..ebae5c3 100644
--- a/build/Toolset.proj
+++ b/build/Toolset.proj
@@ -4,7 +4,8 @@
 
   <PropertyGroup>
     <TargetFramework>net462</TargetFramework>
-    <RestoreSources>https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json</RestoreSources>
+    <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">$(RestoreSources);https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json</RestoreSources>
   </PropertyGroup>
 
   <ItemGroup>
-- 
1.8.3.1

