From 1d6e1710f22bf5c1cd8e71ffa900e57b0addb056 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Tue, 3 Dec 2019 14:05:42 +0000
Subject: [PATCH] Use source-built version of SystemResourcesExtensions

---
 eng/Packages.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Packages.props b/eng/Packages.props
index b45719f..a2dcce8 100644
--- a/eng/Packages.props
+++ b/eng/Packages.props
@@ -32,7 +32,7 @@
     <PackageReference Update="System.Net.Http" Version="4.3.4" />
     <PackageReference Update="System.Reflection.Metadata" Version="1.6.0" />
     <PackageReference Update="System.Reflection.TypeExtensions" Version="4.1.0" />
-    <PackageReference Update="System.Resources.Extensions" Version="4.6.0" />
+    <PackageReference Update="System.Resources.Extensions" Version="$(SystemResourcesExtensionsPackageVersion)" />
     <PackageReference Update="System.Resources.Writer" Version="4.0.0" />
     <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <PackageReference Update="System.Runtime.Loader" Version="4.0.0" />
-- 
1.8.3.1

