From 745b35b282599d451447026c88521794612e1484 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 3 Jun 2019 19:34:39 -0500
Subject: [PATCH 3/3] Import package version props.

---
 eng/Packages.props | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/eng/Packages.props b/eng/Packages.props
index bbf36312..c84fcd7d 100644
--- a/eng/Packages.props
+++ b/eng/Packages.props
@@ -8,6 +8,8 @@
       <NuGetProtocolVersion Condition="'$(NuGetProtocolVersion)' == ''">$(NuGetPackageVersion)</NuGetProtocolVersion>
   </PropertyGroup>
 
+  <Import Project="$(DotNetRestoreSourcePropsPath)" />
+
   <ItemGroup>
     <PackageReference Update="LargeAddressAware" Version="1.0.3" />
     <PackageReference Update="Microsoft.Build.NuGetSdkResolver" Version="$(NuGetBuildTasksVersion)" />
@@ -56,4 +58,4 @@
     <GlobalPackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.15" PrivateAssets="All" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

