From 9ee2fee98b8e01614c52e63b42b2676a1d88d72f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 4 Feb 2019 15:46:21 -0600
Subject: [PATCH] Update SDK version.

---
 eng/Versions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index a5228b6b..de2e7aac 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -38,7 +38,7 @@
 
   <!-- Toolset Dependencies -->
   <PropertyGroup>
-    <DotNetCliVersion>2.1.401</DotNetCliVersion>
+    <DotNetCliVersion>2.1.503</DotNetCliVersion>
     <VSWhereVersion>2.2.7</VSWhereVersion>
   </PropertyGroup>
 
-- 
2.18.0

