From 1401111d92c3de48a684d344fdb1d9770044ba6a Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 3 Jun 2019 19:34:39 -0500
Subject: [PATCH 3/3] Don't overwrite source-build's MS.NETCore.Compilers
 version.

---
 eng/Packages.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/eng/Packages.props b/eng/Packages.props
index bbf36312..952900d6 100644
--- a/eng/Packages.props
+++ b/eng/Packages.props
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-      <MicrosoftNETCoreCompilersVersion>2.10.0</MicrosoftNETCoreCompilersVersion>
+      <MicrosoftNETCoreCompilersVersion Condition="'$(MicrosoftNETCoreCompilersVersion)' == ''">2.10.0</MicrosoftNETCoreCompilersVersion>
       <NuGetPackageVersion Condition="'$(NuGetPackageVersion)' == ''">4.8.0-rtm.5362</NuGetPackageVersion>
       <NuGetBuildTasksVersion Condition="'$(NuGetBuildTasksVersion)' == ''">$(NuGetPackageVersion)</NuGetBuildTasksVersion>
       <NuGetCommandsVersion Condition="'$(NuGetCommandsVersion)' == ''">$(NuGetPackageVersion)</NuGetCommandsVersion>
@@ -56,4 +56,4 @@
     <GlobalPackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.15" PrivateAssets="All" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

