From 1617a9550b61e147e720034c876d7191c30cb98a Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 6 Feb 2019 18:47:59 -0600
Subject: [PATCH] Remove CentralPackageVersions.

---
 Directory.Build.targets | 1 -
 global.json             | 3 +--
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/Directory.Build.targets b/Directory.Build.targets
index 184e2609..ade21e64 100644
--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -11,6 +11,5 @@
   <PropertyGroup>
     <CentralPackagesFile>$(MSBuildThisFileDirectory)eng/Packages.props</CentralPackagesFile>
   </PropertyGroup>
-  <Import Project="Sdk.targets" Sdk="Microsoft.Build.CentralPackageVersions" />
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 </Project>
diff --git a/global.json b/global.json
index 74913f79..f484931e 100644
--- a/global.json
+++ b/global.json
@@ -6,7 +6,6 @@
     }
   },
   "msbuild-sdks": {
-    "Microsoft.Build.CentralPackageVersions": "2.0.1",
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19052.1"
   }
-}
\ No newline at end of file
+}
-- 
2.18.0

