From c22a708dba386b5b0b8ceb618ee5a082eb709762 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 14 Mar 2019 13:56:48 -0500
Subject: [PATCH] Arcade removed this property.

---
 eng/Packages.props | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/eng/Packages.props b/eng/Packages.props
index f3411843..da789b0b 100644
--- a/eng/Packages.props
+++ b/eng/Packages.props
@@ -1,6 +1,9 @@
 <Project>
 
   <PropertyGroup>
+      <MicrosoftNetCompilersVersion>2.8.0</MicrosoftNetCompilersVersion>
+      <MicrosoftNETCoreCompilersVersion>2.10.0</MicrosoftNETCoreCompilersVersion>
+      <MicrosoftNetCompilersToolsetVersion>$(MicrosoftNETCoreCompilersVersion)</MicrosoftNetCompilersToolsetVersion>
       <NuGetPackageVersion Condition="'$(NuGetPackageVersion)' == ''">4.9.0-rtm.5658</NuGetPackageVersion>
       <NuGetBuildTasksVersion Condition="'$(NuGetBuildTasksVersion)' == ''">$(NuGetPackageVersion)</NuGetBuildTasksVersion>
       <NuGetCommandsVersion Condition="'$(NuGetCommandsVersion)' == ''">$(NuGetPackageVersion)</NuGetCommandsVersion>
@@ -54,4 +57,4 @@
     <GlobalPackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.15" PrivateAssets="All" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

