From 5150c74891fe7b57a0ce332326caab087e4398d4 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Wed, 13 Nov 2019 16:31:47 +0000
Subject: [PATCH 3/5] Import PackageVersions.props

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 31bae2cc9..db7aa9ce9 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -142,4 +142,7 @@
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
     <XunitVersion>2.4.1</XunitVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
 </Project>
-- 
2.21.0

