From 0ba59aca7eb538cacfc603ccf3f0e0c283603e32 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Wed, 13 Nov 2019 16:31:47 +0000
Subject: [PATCH 3/4] Import PackageVersions.props

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index dd53e83..7355673 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -141,4 +141,7 @@
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
     <XunitVersion>2.4.1</XunitVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
 </Project>
-- 
2.7.4

