From 4056ed9ce4115fb5db22c5fc088bda17e8d2d3ee Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Wed, 13 Nov 2019 16:31:47 +0000
Subject: [PATCH 3/5] Import PackageVersions.props

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index a3a95fb..eee5f55 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -139,4 +139,7 @@
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
     <XunitVersion>2.4.1</XunitVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
 </Project>
-- 
1.8.3.1

