From 6bf26950f86648b603f0e713c5b7b084e74234c0 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 11 Jul 2019 17:11:07 -0500
Subject: [PATCH 3/3] Add NuGet.org to package sources.

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index cc00f57e..38c13d6a 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -35,6 +35,7 @@
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
+      https://api.nuget.org/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json;
       https://dotnet.myget.org/F/roslyn/api/v3/index.json;
       https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json;
-- 
2.21.0

