From b22d0b1a2568ec0a5a149a6ea252e94c61bb06c8 Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Wed, 13 Nov 2019 16:31:47 +0000
Subject: [PATCH 3/5] Import PackageVersions.props

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 1515c4582..f0fa8e3b9 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -142,4 +142,7 @@
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
     <XunitVersion>2.4.1</XunitVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
 </Project>
-- 
2.18.0

