From 2273d8a159cfe3b07c4b77d4e5a6b601eccaabc7 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 5 Mar 2019 15:52:32 -0600
Subject: [PATCH] Add PackageProjectUrl to avoid new SDK warning.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 6e8b73a..e213431 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -10,5 +10,6 @@
   <PropertyGroup>
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
+    <PackageProjectUrl>https://github.com/dotnet/clicommandlineparser</PackageProjectUrl>
   </PropertyGroup>
 </Project>
-- 
2.18.0

