From 83d9cbffdd20cf0ee1abee86cf84b3a1028173c9 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 31 Mar 2019 00:34:18 -0500
Subject: [PATCH] Skip warning for licenseUrl.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index b1d15db..785547b 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -4,5 +4,6 @@
   <PropertyGroup>
     <RepositoryUrl>https://github.com/dotnet/cli-migrate</RepositoryUrl>
     <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
   </PropertyGroup>
 </Project>
-- 
1.8.3.1

