From f3ade2a647ddd79cf17587dbe8460ea71d149334 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sat, 30 Mar 2019 17:45:56 -0500
Subject: [PATCH] Add PackageProjectUrl for new SDK.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index bc1d365..964b568 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -11,5 +11,6 @@
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
+    <PackageProjectUrl>https://github.com/dotnet/cli-migrate</PackageProjectUrl>
   </PropertyGroup>
 </Project>
-- 
1.8.3.1

