From 5dc78dba33893a894e4975be3bc78518955c1b32 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 22 Jun 2017 11:02:14 -0700
Subject: [PATCH] Unify Versions

---
 build/DependencyVersions.props | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index e116efb..3e5d4e2 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_Migrate_NuGet_Version>4.3.0-preview3-4146</CLI_Migrate_NuGet_Version>
-    <PlatformAbstractionsVersion>1.0.3</PlatformAbstractionsVersion>
-    <DependencyModelVersion>1.0.3</DependencyModelVersion>
+    <CLI_Migrate_NuGet_Version>4.3.0-rtm-4324</CLI_Migrate_NuGet_Version>
+    <PlatformAbstractionsVersion>2.0.0</PlatformAbstractionsVersion>
+    <DependencyModelVersion>2.0.0</DependencyModelVersion>
 
   </PropertyGroup>
 
-- 
1.8.3.1

