From 3175043b88b8f0ec3ad9728789b806116626e503 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 22 Jun 2017 11:02:14 -0700
Subject: [PATCH] Bump NuGet Version

---
 build/DependencyVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index e116efb..22e8df2 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_Migrate_NuGet_Version>4.3.0-preview3-4146</CLI_Migrate_NuGet_Version>
+    <CLI_Migrate_NuGet_Version>4.3.0-preview3-4168</CLI_Migrate_NuGet_Version>
     <PlatformAbstractionsVersion>1.0.3</PlatformAbstractionsVersion>
     <DependencyModelVersion>1.0.3</DependencyModelVersion>
 
-- 
1.8.3.1

