From e59af44d3cc7df4918f5b51c814b91d0a808d77f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 1 May 2019 00:24:47 -0500
Subject: [PATCH] Add LicenseExpression for new warning.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 964b5685..de9a94ff 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -12,5 +12,6 @@
     <DebugSymbols>true</DebugSymbols>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <PackageProjectUrl>https://github.com/dotnet/cli-migrate</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
   </PropertyGroup>
 </Project>
-- 
2.18.0

