From 0e57b7c7697e1b694b75b5db98e056e8b1ac2bc5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 23 Oct 2019 21:14:21 -0500
Subject: [PATCH 1/8] Don't call dotnet without a path

---
 eng/common/tools.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 94965a8fd..8e14f8b6f 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -362,9 +362,9 @@ function MSBuild {
 
     # Work around issues with Azure Artifacts credential provider
     # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
+#    if [[ "$ci" == true ]]; then
+#      dotnet nuget locals http-cache -c
+#    fi
 
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
-- 
2.18.2

