From 950a162091c3562e8a2acc5734e237f611171135 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 24 Jan 2019 13:07:40 -0600
Subject: [PATCH] Transfer patch from CLI.

---
 src/redist/targets/GenerateBundledVersions.targets | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/redist/targets/GenerateBundledVersions.targets b/src/redist/targets/GenerateBundledVersions.targets
index 39f560647..7320b2eda 100644
--- a/src/redist/targets/GenerateBundledVersions.targets
+++ b/src/redist/targets/GenerateBundledVersions.targets
@@ -229,6 +229,14 @@ Copyright (c) .NET Foundation. All rights reserved.
                               RuntimePackRuntimeIdentifiers="@(AspNetCoreRuntimePackRids, '%3B')"
                               />
   </ItemGroup>
+  <ItemGroup>
+    <ImplicitPackageReferenceVersion Update="Microsoft.AspNetCore.App">
+      <DefaultVersion>%25(LatestVersion)</DefaultVersion>
+    </ImplicitPackageReferenceVersion>
+    <ImplicitPackageReferenceVersion Update="Microsoft.AspNetCore.All">
+      <DefaultVersion>%25(LatestVersion)</DefaultVersion>
+    </ImplicitPackageReferenceVersion>
+  </ItemGroup>
 </Project>
 ]]>
     </BundledVersionsPropsContent>
-- 
2.18.0

