From 16dea807496a73adae13a71551242dc39da4415e Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 6 Aug 2018 11:34:32 -0500
Subject: [PATCH] Remove source-build workaround for CoreSetupRid as it is no
 longer needed.

---
 build/BundledRuntimes.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/BundledRuntimes.props b/build/BundledRuntimes.props
index 2de7068b0..8958a919c 100644
--- a/build/BundledRuntimes.props
+++ b/build/BundledRuntimes.props
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CoreSetupRid>$(HostRid)</CoreSetupRid>
-    <CoreSetupRid Condition=" ('$(HostOSName)' == 'win' or '$(HostOSName)' == 'osx') and '$(DotNetBuildFromSource)' != 'true' ">$(HostMonikerRid)</CoreSetupRid>
+    <CoreSetupRid Condition=" '$(HostOSName)' == 'win' or '$(HostOSName)' == 'osx' ">$(HostMonikerRid)</CoreSetupRid>
 
     <!-- only the runtime OSX .pkgs have a `-internal` suffix -->
     <InstallerStartSuffix Condition="'$(HostOSName)' == 'osx'">-internal</InstallerStartSuffix>
-- 
2.14.1

