From 1f8c14f7247546db4cb433930f09391b82f412e7 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 6 Feb 2019 12:23:22 -0600
Subject: [PATCH 3/5] Remove test projects from solution.

---
 Microsoft.DotNet.Cli.sln | 43 +++++++++++++++++++---------------------
 1 file changed, 20 insertions(+), 23 deletions(-)

diff --git a/Microsoft.DotNet.Cli.sln b/Microsoft.DotNet.Cli.sln
index 0b8150d41..4f6865008 100644
--- a/Microsoft.DotNet.Cli.sln
+++ b/Microsoft.DotNet.Cli.sln
@@ -5,18 +5,10 @@ VisualStudioVersion = 16.0.28603.18
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{ED2FE3E2-F7E7-4389-8231-B65123F2076F}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{17735A9D-BFD9-4585-A7CB-3208CA6EA8A7}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "redist", "src\redist\redist.csproj", "{098D9321-1201-4974-A75E-F58EBCD98ACF}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EndToEnd.Tests", "test\EndToEnd\EndToEnd.Tests.csproj", "{1BFF54F9-4E35-49DB-893C-AF2047722FC6}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "core-sdk-tasks", "src\core-sdk-tasks\core-sdk-tasks.csproj", "{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Tools.Tests.Utilities", "test\Microsoft.DotNet.Tools.Tests.Utilities\Microsoft.DotNet.Tools.Tests.Utilities.csproj", "{78E15EC1-7732-41E3-8591-934E9F583254}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SdkResolver", "src\SdkResolver\SdkResolver.csproj", "{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0}"
-EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -27,32 +19,37 @@ Global
 		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{1BFF54F9-4E35-49DB-893C-AF2047722FC6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{1BFF54F9-4E35-49DB-893C-AF2047722FC6}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{1BFF54F9-4E35-49DB-893C-AF2047722FC6}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{1BFF54F9-4E35-49DB-893C-AF2047722FC6}.Release|Any CPU.Build.0 = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|x64.ActiveCfg = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|x64.Build.0 = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|x86.ActiveCfg = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.Release|x86.Build.0 = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|Any CPU.ActiveCfg = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|Any CPU.Build.0 = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|x64.ActiveCfg = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|x64.Build.0 = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|x86.ActiveCfg = Release|Any CPU
+		{098D9321-1201-4974-A75E-F58EBCD98ACF}.RelWithDebInfo|x86.Build.0 = Release|Any CPU
 		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|Any CPU.Build.0 = Release|Any CPU
-		{78E15EC1-7732-41E3-8591-934E9F583254}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{78E15EC1-7732-41E3-8591-934E9F583254}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{78E15EC1-7732-41E3-8591-934E9F583254}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{78E15EC1-7732-41E3-8591-934E9F583254}.Release|Any CPU.Build.0 = Release|Any CPU
-		{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0}.Release|Any CPU.Build.0 = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|x64.ActiveCfg = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|x64.Build.0 = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|x86.ActiveCfg = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.Release|x86.Build.0 = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|Any CPU.ActiveCfg = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|Any CPU.Build.0 = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|x64.ActiveCfg = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|x64.Build.0 = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|x86.ActiveCfg = Release|Any CPU
+		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C}.RelWithDebInfo|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
 		{098D9321-1201-4974-A75E-F58EBCD98ACF} = {ED2FE3E2-F7E7-4389-8231-B65123F2076F}
-		{1BFF54F9-4E35-49DB-893C-AF2047722FC6} = {17735A9D-BFD9-4585-A7CB-3208CA6EA8A7}
 		{53AF2D01-B69F-4CD0-86A7-8FD95967D23C} = {ED2FE3E2-F7E7-4389-8231-B65123F2076F}
-		{78E15EC1-7732-41E3-8591-934E9F583254} = {17735A9D-BFD9-4585-A7CB-3208CA6EA8A7}
-		{7EE15292-2CAD-44FA-8A1F-BAC4688A49E0} = {ED2FE3E2-F7E7-4389-8231-B65123F2076F}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {B526D2CE-EE2D-4AD4-93EF-1867D90FF1F5}
-- 
2.18.0

