From bd4c36f815aae849dbce7f500519cf681600a0a1 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 26 Sep 2019 07:44:26 -0500
Subject: [PATCH 4/4] Don't call dotnet without path

---
 eng/common/tools.sh | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 94965a8fd..629dbb583 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -360,12 +360,6 @@ function MSBuild {
     InitializeBuildTool
     InitializeToolset
 
-    # Work around issues with Azure Artifacts credential provider
-    # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
-
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
     args=( "${args[@]}" "-logger:$logger_path" )
-- 
2.18.0

