From 2f44d27f3007d995b3b125ec8c1ce4255cc53ef5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Sun, 17 Feb 2019 08:31:22 -0600
Subject: [PATCH 4/5] Add PVP import

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 35db136e9..4b703a262 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -69,6 +69,9 @@
     <MicrosoftNETTestSdkVersion>15.8.0</MicrosoftNETTestSdkVersion>
     <MicrosoftSourceLinkVersion>1.0.0-beta2-18618-05</MicrosoftSourceLinkVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
   <PropertyGroup>
     <RestoreSources>
       $(RestoreSources);
-- 
2.18.0

