From 5bbd4a68be05d506fe517c175717f23f22015176 Mon Sep 17 00:00:00 2001
From: Tom Deseyn <tom.deseyn@gmail.com>
Date: Thu, 16 Jan 2020 08:37:34 +0100
Subject: [PATCH] Add target rid to AppHostRuntimeIdentifiers

---
 src/redist/targets/GenerateBundledVersions.targets | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/redist/targets/GenerateBundledVersions.targets b/src/redist/targets/GenerateBundledVersions.targets
index b657fdb..db99761 100644
--- a/src/redist/targets/GenerateBundledVersions.targets
+++ b/src/redist/targets/GenerateBundledVersions.targets
@@ -60,6 +60,11 @@
       <NetCore31RuntimePackRids Include="@(NetCore30RuntimePackRids)"/>
       <NetCoreRuntimePackRids Include="@(NetCore31RuntimePackRids)"/>
 
+      <NetCoreAppHostPackRids Include="
+          $(ProductMonikerRid);
+          @(NetCoreRuntimePackRids)
+          " />
+
       <AspNetCore30RuntimePackRids Include="
         win-x64;
         win-x86;
@@ -183,7 +188,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                       TargetFramework="netcoreapp5.0"
                       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
                       AppHostPackVersion="$(_NETCoreAppPackageVersion)"
-                      AppHostRuntimeIdentifiers="@(NetCoreRuntimePackRids, '%3B')"
+                      AppHostRuntimeIdentifiers="@(NetCoreAppHostPackRids, '%3B')"
                       />
 
     <KnownCrossgen2Pack Include="Microsoft.NETCore.App.Crossgen2"
-- 
1.8.3.1

