From 43e76ed4b365181bcff81b54a08500026ab4c82a Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 24 Jun 2019 14:04:28 -0500
Subject: [PATCH 7/7] We shouldn't be using dotnet-deb-tool in source-build.

---
 src/redist/targets/GenerateDebs.targets | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/redist/targets/GenerateDebs.targets b/src/redist/targets/GenerateDebs.targets
index 5b9254aa2..d15934b7b 100644
--- a/src/redist/targets/GenerateDebs.targets
+++ b/src/redist/targets/GenerateDebs.targets
@@ -1,11 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <!-- dotnet deb-tool -->
-  <ItemGroup>
-    <DotNetCliToolReference Include="dotnet-deb-tool" Version="$(DotnetDebToolVersion)" />
-  </ItemGroup>
-  
   <Target Name="SetupDebProps"
         DependsOnTargets="SetSdkVersionInfo;
                           GetCurrentRuntimeInformation;
-- 
2.21.0

