From 268484d1d4738d334381ebf04c33cf393e811668 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 4 Dec 2019 21:53:35 +0000
Subject: [PATCH 7/8] Don't set SourceLinkVersion in core-sdk

---
 eng/Versions.props | 1 -
 1 file changed, 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 886382ecc..2d7f79556 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -117,7 +117,6 @@
     <VersionToolsVersion>$(BuildTasksFeedToolVersion)</VersionToolsVersion>
     <DotnetDebToolVersion>2.0.0</DotnetDebToolVersion>
     <MicrosoftNETTestSdkVersion>15.8.0</MicrosoftNETTestSdkVersion>
-    <MicrosoftSourceLinkVersion>1.0.0-beta2-18618-05</MicrosoftSourceLinkVersion>
   </PropertyGroup>
   <PropertyGroup>
     <!-- pinned dependency. This package is not being produced outside of the 2.0 branch of corefx and should not change. -->
-- 
2.18.2

