From e539136f37b6d8a53c6b518e2f01e574d2f16026 Mon Sep 17 00:00:00 2001
From: Tom Deseyn <tom.deseyn@gmail.com>
Date: Thu, 16 Jan 2020 08:37:34 +0100
Subject: [PATCH 8/8] Add target rid to AppHostRuntimeIdentifiers

---
 src/redist/targets/GenerateBundledVersions.targets | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/redist/targets/GenerateBundledVersions.targets b/src/redist/targets/GenerateBundledVersions.targets
index a214842a0..2c350f15f 100644
--- a/src/redist/targets/GenerateBundledVersions.targets
+++ b/src/redist/targets/GenerateBundledVersions.targets
@@ -48,6 +48,11 @@
 
       <NetCoreRuntimePackRids Include="@(NetCore30RuntimePackRids)"/>
 
+      <NetCoreAppHostPackRids Include="
+          $(ProductMonikerRid);
+          @(NetCoreRuntimePackRids)
+          " />
+
       <AspNetCore30RuntimePackRids Include="
         win-x64;
         win-x86;
@@ -168,7 +173,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                       TargetFramework="netcoreapp3.1"
                       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
                       AppHostPackVersion="$(_NETCoreAppPackageVersion)"
-                      AppHostRuntimeIdentifiers="@(NetCoreRuntimePackRids, '%3B')"
+                      AppHostRuntimeIdentifiers="@(NetCoreAppHostPackRids, '%3B')"
                       />
     
     <KnownFrameworkReference Include="Microsoft.WindowsDesktop.App"
-- 
2.18.2

