From 2f0f2f9a5f9210ef6386eb194e31d10b1bd2f3f5 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Sat, 20 May 2017 16:11:18 -0700
Subject: [PATCH] Add BuildForCli target

---
 build/build.proj | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/build/build.proj b/build/build.proj
index 12377f5..0803709 100644
--- a/build/build.proj
+++ b/build/build.proj
@@ -76,7 +76,22 @@
              Targets="Pack"
              Properties="$(CommonMSBuildGlobalProperties);NoBuild=true"
              />
+  </Target>
+
+  <Target Name="BuildForCli">
+    <MSBuild Projects="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\Microsoft.NET.Build.Tasks.csproj"
+             Targets="Restore"
+             Properties="$(CommonMSBuildGlobalProperties)" />
 
+    <MSBuild Projects="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\Microsoft.NET.Build.Tasks.csproj"
+             Targets="Build"
+             Properties="$(CommonMSBuildGlobalProperties)" />
+
+    <MSBuild BuildInParallel="true"
+             Projects="@(NuGetProjectFile)"
+             Targets="Pack"
+             Properties="$(CommonMSBuildGlobalProperties);NoBuild=true"
+             />
   </Target>
   
   <Target Name="CleanNuGetPackages">
-- 
1.8.3.1

