From 06d3c0dbd226c3f7cd7aafd89ccf2e42f15dfce8 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Wed, 31 Oct 2018 23:04:45 -0500
Subject: [PATCH] Exclude unit test projects from source-build

---
 src/Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Directory.Build.props b/src/Directory.Build.props
index ca3ebe94..396d7b21 100644
--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -29,6 +29,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <GenerateProgramFile>false</GenerateProgramFile>
     <XUnitRunnerAdditionalArguments>-parallel none</XUnitRunnerAdditionalArguments>
-- 
2.17.1.windows.2

