From 805cc7d8262bbe0236758473d2fc6bf7db9dca6a Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Fri, 6 Apr 2018 15:51:01 -0500
Subject: [PATCH] Remove --log option from defaults

---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index 9816b123..ea63e00d 100755
--- a/build.sh
+++ b/build.sh
@@ -8,4 +8,4 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 ScriptRoot="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-. "$ScriptRoot/build/build.sh" --build --restore --log "$@"
+. "$ScriptRoot/build/build.sh" --build --restore "$@"
-- 
2.16.1.windows.4

