From 11bc15f7eab75cf18f73d6e4a39c54f2846ab1b9 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Sat, 20 May 2017 16:11:18 -0700
Subject: [PATCH] Add BuildForCli target

---
 build/build.proj | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/build/build.proj b/build/build.proj
index 7794fe0..27eadec 100644
--- a/build/build.proj
+++ b/build/build.proj
@@ -78,7 +78,22 @@
              Targets="Pack"
              Properties="$(CommonMSBuildGlobalProperties);NoBuild=true"
              />
+  </Target>
+
+  <Target Name="BuildForCli">
+    <MSBuild Projects="@(NuGetProjectFile)"
+             Targets="Restore"
+             Properties="$(CommonMSBuildGlobalProperties)" />
 
+    <MSBuild Projects="@(NuGetProjectFile)"
+             Targets="Build"
+             Properties="$(CommonMSBuildGlobalProperties)" />
+
+    <MSBuild BuildInParallel="true"
+             Projects="@(NuGetProjectFile)"
+             Targets="Pack"
+             Properties="$(CommonMSBuildGlobalProperties);NoBuild=true"
+             />
   </Target>
   
   <Target Name="CleanNuGetPackages">
-- 
1.8.3.1

