From cec70816c3681c3492ac32c4a94d51af979b0c6a Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 15 Jun 2017 16:02:24 -0700
Subject: [PATCH] Bump NuGet Version

---
 build/DependencyVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index 002bb9b..4984442 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -10,7 +10,7 @@
     <MsBuildPackagesVersion>15.2.0-preview-000093-02</MsBuildPackagesVersion>
     <DependencyModelVersion>2.0.0-preview1-001960</DependencyModelVersion>
     <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>
-    <NuGetVersion>4.3.0-preview2-4095</NuGetVersion>
+    <NuGetVersion>4.3.0-preview3-4146</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
   </PropertyGroup>
-- 
1.8.3.1

