From 7bdc7dc9dbd58932639bf490a970e052e4966fd6 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Fri, 21 Jul 2017 09:13:52 -0700
Subject: [PATCH] Unify versions

---
 build/DependencyVersions.props | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index 0e78565..ba8d754 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -7,13 +7,13 @@
   </PropertyGroup>
   
   <PropertyGroup>
-    <MsBuildPackagesVersion>15.2.0-preview-000093-02</MsBuildPackagesVersion>
-    <DependencyModelVersion>2.0.0-preview1-001960</DependencyModelVersion>
-    <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>
+    <MsBuildPackagesVersion>15.3.409</MsBuildPackagesVersion>
+    <DependencyModelVersion>2.0.0</DependencyModelVersion>
+    <PlatformAbstractionsVersion>2.0.0</PlatformAbstractionsVersion>
     <NuGetVersion>4.3.0-rtm-4324</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview3-25519-03</NETStandardLibraryNETFrameworkVersion>
     <XliffTasksVersion>0.2.0-beta-000042</XliffTasksVersion>
   </PropertyGroup>
 
-- 
1.8.3.1

