From a8ade788c1be47099a533583a855ba16f0f12420 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Thu, 15 Jun 2017 16:21:29 -0700
Subject: [PATCH] Bump MSBuild Version

---
 build/DependencyVersions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index 4984442..3710661 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -7,7 +7,7 @@
   </PropertyGroup>
   
   <PropertyGroup>
-    <MsBuildPackagesVersion>15.2.0-preview-000093-02</MsBuildPackagesVersion>
+    <MsBuildPackagesVersion>15.3.0-preview-000378-01</MsBuildPackagesVersion>
     <DependencyModelVersion>2.0.0-preview1-001960</DependencyModelVersion>
     <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>
     <NuGetVersion>4.3.0-preview3-4146</NuGetVersion>
-- 
1.8.3.1

