From 001359ea78e2649ec81289a32ca7cae71542d872 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 26 Sep 2019 07:41:22 -0500
Subject: [PATCH 3/4] Don't call dotnet without path

---
 eng/common/tools.sh | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 757d5b9..a8fabf5 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -338,12 +338,6 @@ function MSBuild {
     InitializeBuildTool
     InitializeToolset
 
-    # Work around issues with Azure Artifacts credential provider
-    # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
-
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
     args=( "${args[@]}" "-logger:$logger_path" )
-- 
1.8.3.1

