From 0f85807b2c27e881bfb0f367a8c002fb318a6573 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 8 Sep 2020 12:20:40 -0500
Subject: [PATCH 11/11] Use source-built MSBuild for F#.

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index bdacff7..ec1efc4 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -92,6 +92,7 @@
     <MicrosoftBuildVersion>$(MicrosoftBuildPackageVersion)</MicrosoftBuildVersion>
     <MicrosoftBuildFrameworkVersion>$(MicrosoftBuildFrameworkPackageVersion)</MicrosoftBuildFrameworkVersion>
     <MicrosoftBuildUtilitiesCoreVersion>$(MicrosoftBuildUtilitiesCorePackageVersion)</MicrosoftBuildUtilitiesCoreVersion>
+    <FSharpBuildVersion>$(MicrosoftBuildPackageVersion)</FSharpBuildVersion>
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/dotnet/CliCommandLineParser -->
-- 
2.18.0

