From 36ef4474f2d76aaee43594e90aa010f9b9e977a7 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 1 Feb 2021 15:21:00 -0500
Subject: [PATCH] Add project references in redist.csproj

---
 src/Layout/redist/redist.csproj | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/Layout/redist/redist.csproj b/src/Layout/redist/redist.csproj
index 4675f37..bc77403 100644
--- a/src/Layout/redist/redist.csproj
+++ b/src/Layout/redist/redist.csproj
@@ -50,6 +50,16 @@
     <ProjectReference Include="..\..\Tasks\Microsoft.NET.Build.Extensions.Tasks\Microsoft.NET.Build.Extensions.Tasks.csproj"
                       ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\..\Resolvers\Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver\Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.csproj" />
+
+    <ProjectReference Include="..\..\WebSdk\ProjectSystem\Tasks\Microsoft.NET.Sdk.Web.ProjectSystem.Tasks.csproj"
+                      ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\WebSdk\Publish\Tasks\Microsoft.NET.Sdk.Publish.Tasks.csproj"
+                      ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\WebSdk\Web\Tasks\Microsoft.NET.Sdk.Web.Tasks.csproj"
+                      ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\WebSdk\Worker\Tasks\Microsoft.NET.Sdk.Worker.Tasks.csproj"
+                      ReferenceOutputAssembly="false" />
+
   </ItemGroup>
 
   <ItemGroup>
-- 
1.8.3.1

