From ecac294b9fc41a2ada9dffa7654724914f4c2df2 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 12 Mar 2021 14:58:14 -0600
Subject: [PATCH 9/9] Fix bootstrapping - this property was removed.

---
 src/redist/redist.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
index 303925ad9..2dbb7447c 100644
--- a/src/redist/redist.csproj
+++ b/src/redist/redist.csproj
@@ -27,7 +27,7 @@
   <Import Project="targets\Versions.targets" />
   <Import Project="targets\Branding.targets" />
   <Import Project="targets\BundledTemplates.targets" />
-  <Import Project="targets\BundledDotnetTools.targets" />
+  <Import Project="targets\BundledDotnetTools.targets" Condition="'$(CLIBUILD_SKIP_BUNDLEDDOTNETTOOLS)' != 'true'" />
   <Import Project="targets\GenerateBundledVersions.targets" />
   <Import Project="targets\Crossgen.targets" />
   <Import Project="targets\GenerateLayout.targets" />
-- 
2.18.0

