From 1ebd5a2a76a4a563a5336b7fd8acde9326db095f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 6 Mar 2018 17:25:10 -0600
Subject: [PATCH] Add solution for only .NET Core projects.

---
 RoslynToolsNetCore.sln | 58 ++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 58 insertions(+)
 create mode 100644 RoslynToolsNetCore.sln

diff --git a/RoslynToolsNetCore.sln b/RoslynToolsNetCore.sln
new file mode 100644
index 0000000..69299ab
--- /dev/null
+++ b/RoslynToolsNetCore.sln
@@ -0,0 +1,58 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27121.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RepoToolset", "sdks\RepoToolset\RepoToolset.csproj", "{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SDKs", "SDKs", "{57FAEEA0-DEBE-4939-85C9-FBDF07428CCB}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuGetTools", "NuGetTools", "{732A897A-76AE-41E3-A7E4-55F16C1D56EB}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RoslynTools.BuildTasks", "src\BuildTasks\RoslynTools.BuildTasks.csproj", "{995A02BA-FDE6-46E2-9983-2FAA299676DD}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|x64.Build.0 = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Debug|x86.Build.0 = Debug|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|Any CPU.Build.0 = Release|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|x64.ActiveCfg = Release|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|x64.Build.0 = Release|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|x86.ActiveCfg = Release|Any CPU
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4}.Release|x86.Build.0 = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|x64.Build.0 = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Debug|x86.Build.0 = Debug|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|Any CPU.Build.0 = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|x64.ActiveCfg = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|x64.Build.0 = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|x86.ActiveCfg = Release|Any CPU
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD}.Release|x86.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{C01BFB87-04AD-4DA1-B3FB-1DCDB1FDBBD4} = {57FAEEA0-DEBE-4939-85C9-FBDF07428CCB}
+		{995A02BA-FDE6-46E2-9983-2FAA299676DD} = {57FAEEA0-DEBE-4939-85C9-FBDF07428CCB}
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {738CC66D-49E2-4B21-835F-94170D22A29D}
+	EndGlobalSection
+EndGlobal
-- 
2.14.1

