From bb464a57817f07a3dbf9e3b888f3e22abdd15a15 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 23 Oct 2019 20:04:55 -0500
Subject: [PATCH 01/15] Don't call dotnet without path

---
 eng/common/tools.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 6221830226..d3635b8c65 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -370,9 +370,9 @@ function MSBuild {
 
     # Work around issues with Azure Artifacts credential provider
     # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
+#    if [[ "$ci" == true ]]; then
+#      dotnet nuget locals http-cache -c
+#    fi
 
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
-- 
2.21.0

