From 6d2af64a604592102fc6300d7291816703878b64 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 17 Jul 2019 01:39:32 -0500
Subject: [PATCH] Add additional needed feed.

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 3d796aacf9..85785be9c1 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -254,6 +254,7 @@
       https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json;
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://grpc.jfrog.io/grpc/api/nuget/v3/grpc-nuget-dev;
+      https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json;
     </RestoreSources>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true' AND $(MicrosoftNetCompilersToolsetPackageVersion.Contains('-')) ">
       $(RestoreSources);
-- 
2.18.0

