From 14eaf2a7cd29965ead64dd61b40ae87b216b36c9 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 23 Oct 2019 20:04:55 -0500
Subject: [PATCH 1/8] Don't call dotnet without path

---
 eng/common/tools.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 757d5b9..5a4cfeb 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -340,9 +340,9 @@ function MSBuild {
 
     # Work around issues with Azure Artifacts credential provider
     # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
+#    if [[ "$ci" == true ]]; then
+#      dotnet nuget locals http-cache -c
+#    fi
 
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
-- 
1.8.3.1

