From d77fea1bc8a05e9f5cad5cfaf37c8fd426cd58f4 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 17 Jul 2019 17:20:48 -0500
Subject: [PATCH 02/10] Import PackageVersions.props.

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 27cfc944d3..29845aa4b1 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -264,6 +264,9 @@
     <XunitExtensibilityExecutionPackageVersion>$(XunitVersion)</XunitExtensibilityExecutionPackageVersion>
     <MicrosoftDataSqlClientPackageVersion>1.0.19249.1</MicrosoftDataSqlClientPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
   <!-- Restore feeds -->
   <PropertyGroup Label="Restore feeds">
     <!-- In an orchestrated build, this may be overridden to other Azure feeds. -->
-- 
2.18.0

