From 166f18cb451a3e6a00676cb918b8ef16a92fe99a Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 15 May 2020 16:38:48 +0000
Subject: [PATCH 03/11] Exclude analyzer for source-build

---
 Directory.Build.props | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 21ae56027b..86dbc3c929 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -146,7 +146,8 @@
       AND '$(IsBenchmarkProject)' != 'true'
       AND '$(IsTestAssetProject)' != 'true'
       AND '$(MSBuildProjectName)' != 'Internal.AspNetCore.Analyzers'
-      AND '$(MSBuildProjectExtension)' == '.csproj'">
+      AND '$(MSBuildProjectExtension)' == '.csproj'
+      AND '$(DotNetBuildFromSource)' != 'true'">
     <ProjectReference
       Include="$(RepoRoot)src\Analyzers\Internal.AspNetCore.Analyzers\src\Internal.AspNetCore.Analyzers.csproj"
       ReferenceOutputAssembly="false"
-- 
2.18.2

