From 36db65e3e4a7e141fc11875d06705ca1c4f79031 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Thu, 31 Oct 2019 20:38:26 +0000
Subject: [PATCH 03/12] Conditionally set PackAsToolShimRID

---
 Directory.Build.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Directory.Build.targets b/Directory.Build.targets
index 0e0db95..854c5ea 100644
--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -45,7 +45,7 @@
     <SuppressDependenciesWhenPacking Condition="'$(SuppressDependenciesWhenPacking)' == '' AND '$(IsAnalyzersProject)' == 'true'">true</SuppressDependenciesWhenPacking>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(PackAsTool)' == 'true' AND '$(IsShippingPackage)' == 'true'">
+  <PropertyGroup Condition="'$(PackAsTool)' == 'true' AND '$(IsShippingPackage)' == 'true' AND '$(DotNetBuildFromSource)' != 'true'">
     <!-- This is a requirement for Microsoft tool packages only. -->
     <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86</PackAsToolShimRuntimeIdentifiers>
   </PropertyGroup>
-- 
2.7.4

