From d986243c5e28b7596d7ef62043edd505cdab5f8f Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 18 Jul 2019 03:04:54 -0500
Subject: [PATCH 3/3] Add default for _SuppressSdkImports?

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index d7c5acd6b0..fe73bbdca7 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -3,6 +3,7 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <RepositoryUrl>https://github.com/aspnet/AspNetCore</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
+    <_SuppressSdkImports Condition="'$(_SuppressSdkImports)' == ''">false</_SuppressSdkImports>
   </PropertyGroup>
 
   <!-- Defines project type conventions. -->
-- 
2.18.0

