From dbcf80e6a7439c9454d0a54acd16c2bd94f15e50 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 18 Jul 2019 03:04:54 -0500
Subject: [PATCH 4/4] Add default for _SuppressSdkImports?

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 742784c5e5..a611855d4c 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -3,6 +3,7 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <RepositoryUrl>https://github.com/aspnet/AspNetCore</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
+    <_SuppressSdkImports Condition="'$(_SuppressSdkImports)' == ''">false</_SuppressSdkImports>
   </PropertyGroup>
 
   <!-- Defines project type conventions. -->
-- 
2.21.0

