From 07ba3ad31a72c8bb5a2b9da2fccdbe3fbcc0de86 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 8 Aug 2019 11:52:41 -0500
Subject: [PATCH 5/5] Allow an override RID.

---
 Directory.Build.props | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 1f36384bc5..ef9171bde6 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -112,7 +112,8 @@
       linux-musl-arm64;
       linux-x64;
       linux-arm;
-      linux-arm64
+      linux-arm64;
+      $(TargetRuntimeIdentifier)
     </SupportedRuntimeIdentifiers>
 
     <!-- Make error messages clickable in VS Code's console -->
-- 
2.18.0

