From 7e94952e0a971350d32b4544927e28f6480f129c Mon Sep 17 00:00:00 2001
From: adaggarwal <aditya.aggarwal@microsoft.com>
Date: Thu, 14 Nov 2019 16:55:29 +0000
Subject: [PATCH 05/16] Import PackageVersions.props

---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index d28960c837..c23a40334e 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -265,6 +265,9 @@
     <XunitExtensibilityExecutionPackageVersion>$(XunitVersion)</XunitExtensibilityExecutionPackageVersion>
     <MicrosoftDataSqlClientPackageVersion>1.0.19249.1</MicrosoftDataSqlClientPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
   <!-- Restore feeds -->
   <PropertyGroup Label="Restore feeds">
     <!-- In an orchestrated build, this may be overridden to other Azure feeds. -->
-- 
2.21.0

