From 97c59350f47cdc283299c232d804b890fc6ac109 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 4 Aug 2020 13:36:49 -0500
Subject: [PATCH 05/13] Exclude Analyzers from source-build.

---
 .../src/Internal.AspNetCore.Analyzers.csproj                     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Analyzers/Internal.AspNetCore.Analyzers/src/Internal.AspNetCore.Analyzers.csproj b/src/Analyzers/Internal.AspNetCore.Analyzers/src/Internal.AspNetCore.Analyzers.csproj
index f4d42dc817..c601734f5a 100644
--- a/src/Analyzers/Internal.AspNetCore.Analyzers/src/Internal.AspNetCore.Analyzers.csproj
+++ b/src/Analyzers/Internal.AspNetCore.Analyzers/src/Internal.AspNetCore.Analyzers.csproj
@@ -17,6 +17,7 @@
     <IsShippingPackage>false</IsShippingPackage>
     <!-- This project needs an older version of M.CA.CSharp.Workspaces since it targets netstandard1.3 -->
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.18.0

