From a6e01b60d831063d43e2941735a30df04cd380d3 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 16 Aug 2019 11:44:03 -0500
Subject: [PATCH 6/8] Elevate for ulimit on OSX?

---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index ad4ce2c1c8..b574bc04a7 100755
--- a/build.sh
+++ b/build.sh
@@ -285,7 +285,7 @@ fi
 
 # increase file descriptor limit on macOS
 if [ "$(uname)" = "Darwin" ]; then
-    ulimit -n 10000
+    sudo ulimit -n 10000
 fi
 
 # Import Arcade
-- 
2.18.0

