From fff33ea363b1e4221075cb0378fb3ff9730c1389 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 16 Aug 2019 11:44:03 -0500
Subject: [PATCH 6/6] Elevate for ulimit on OSX?

---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index c170ac1844..a41626f0a7 100755
--- a/build.sh
+++ b/build.sh
@@ -270,7 +270,7 @@ fi
 
 # increase file descriptor limit on macOS
 if [ "$(uname)" = "Darwin" ]; then
-    ulimit -n 10000
+    sudo ulimit -n 10000
 fi
 
 # Import Arcade
-- 
2.18.0

