From bb5b0a745c5ec9ff305c0ca0259e1b15acaab2bf Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Thu, 26 Sep 2019 00:35:59 -0500
Subject: [PATCH 06/10] Don't call out to dotnet with no path

---
 eng/common/tools.sh | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/eng/common/tools.sh b/eng/common/tools.sh
index 6221830226..ee785e1040 100755
--- a/eng/common/tools.sh
+++ b/eng/common/tools.sh
@@ -368,12 +368,6 @@ function MSBuild {
     InitializeBuildTool
     InitializeToolset
 
-    # Work around issues with Azure Artifacts credential provider
-    # https://github.com/dotnet/arcade/issues/3932
-    if [[ "$ci" == true ]]; then
-      dotnet nuget locals http-cache -c
-    fi
-
     local toolset_dir="${_InitializeToolset%/*}"
     local logger_path="$toolset_dir/$_InitializeBuildToolFramework/Microsoft.DotNet.Arcade.Sdk.dll"
     args=( "${args[@]}" "-logger:$logger_path" )
-- 
2.18.0

