From e5afe782c24e764990942deacc8c11f510186429 Mon Sep 17 00:00:00 2001
From: John Luo <johluo@microsoft.com>
Date: Thu, 29 Aug 2019 15:29:28 -0700
Subject: [PATCH 6/6] Build analyzers in source build

---
 Directory.Build.targets | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Directory.Build.targets b/Directory.Build.targets
index 5006951f11..f5b217f365 100644
--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -16,7 +16,8 @@
     <IsPackable Condition="'$(IsAspNetCoreApp)' == 'true' AND '$(IsShippingPackage)' != 'true'">false</IsPackable>
 
     <!-- Only build assemblies in Microsoft.AspNetCore.App in source build -->
-    <ExcludeFromSourceBuild Condition="'$(ExcludeFromSourceBuild)' == '' and '$(DotNetBuildFromSource)' == 'true' and '$(IsAspNetCoreApp)' != 'true'">true</ExcludeFromSourceBuild>
+    <!-- Analyzer package are needed in source build for WebSDK -->
+    <ExcludeFromSourceBuild Condition="'$(ExcludeFromSourceBuild)' == '' and '$(DotNetBuildFromSource)' == 'true' and '$(IsAspNetCoreApp)' != 'true' and '$(IsAnalyzersProject)' != 'true'">true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <PropertyGroup Label="Resx settings">
-- 
2.17.1

