From b1410d803f306572f6ee381df6648f016a445a22 Mon Sep 17 00:00:00 2001
From: John Luo <johluo@microsoft.com>
Date: Fri, 16 Aug 2019 11:25:36 -0700
Subject: [PATCH 7/7] Add a package for source build

---
 .../Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj b/src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj
index b75277126a..922477578a 100644
--- a/src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj
+++ b/src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj
@@ -7,6 +7,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;aspnetcoremvc;razor</PackageTags>
     <IsShippingPackage>true</IsShippingPackage>
+    <!-- Build this package in source build for testing F# templates -->
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <ItemGroup>
-- 
2.18.0

