From 02c8003d8505237a7d12b45d7b96d0324609b0a0 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 9 Oct 2019 19:27:51 +0000
Subject: [PATCH] Update NuGetBuildTasks version to ref pkg version

---
 eng/tools/RepoTasks/RepoTasks.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/tools/RepoTasks/RepoTasks.csproj b/eng/tools/RepoTasks/RepoTasks.csproj
index 3f8c247..ff0cf5b 100644
--- a/eng/tools/RepoTasks/RepoTasks.csproj
+++ b/eng/tools/RepoTasks/RepoTasks.csproj
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.Build.Tasks" Version="5.1.0-rtm.5921" />
+    <PackageReference Include="NuGet.Build.Tasks" Version="5.1.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
   </ItemGroup>
 
-- 
1.8.3.1

