From 577d5fe7cbdc80e4428a3bb146fe8a7e9af803d9 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 1 Nov 2019 14:38:11 +0000
Subject: [PATCH] Remove samples and tests from ProjectToBuild

---
 eng/Build.props | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/eng/Build.props b/eng/Build.props
index fa5e1d2..5e0ebf8 100644
--- a/eng/Build.props
+++ b/eng/Build.props
@@ -119,6 +119,10 @@
 
         <ProjectToBuild Condition=" '$(BuildJava)' == 'true'" Include="@(JavaProjects)" Exclude="@(ProjectToExclude)" />
         <ProjectToExclude Condition=" '$(BuildJava)' != 'true'" Include="@(JavaProjects)" />
+        <ProjectToExclude Condition=" '$(DotNetBuildFromSource)' == 'true'" Include="$(RepoRoot)src\**\samples\**\*.*proj" />
+        <ProjectToExclude Condition=" '$(DotNetBuildFromSource)' == 'true'" Include="$(RepoRoot)src\**\sample\**\*.*proj" />
+        <ProjectToExclude Condition=" '$(DotNetBuildFromSource)' == 'true'" Include="$(RepoRoot)src\**\tests\**\*.*proj" />
+        <ProjectToExclude Condition=" '$(DotNetBuildFromSource)' == 'true'" Include="$(RepoRoot)src\**\test\**\*.*proj" />
 
         <!--
           Use caution to avoid deep recursion. If the globbing pattern picks up something which exceeds MAX_PATH,
-- 
1.8.3.1

