From ddd04cae937e4f8595cf0f93b691a49c896037d6 Mon Sep 17 00:00:00 2001
From: Tomas Weinfurt <tweinfurt@yahoo.com>
Date: Wed, 13 Nov 2019 22:32:02 -0800
Subject: [PATCH 11/16] Add FreeBSD

---
 Directory.Build.props | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index d7691d9b8d..4000049e52 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -108,6 +108,7 @@
   <PropertyGroup>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('Windows'))">win</TargetOsName>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('OSX'))">osx</TargetOsName>
+    <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('FreeBSD'))">freebsd</TargetOsName>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('Linux'))">linux</TargetOsName>
     <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">x64</TargetArchitecture>
     <TargetRuntimeIdentifier>$(TargetOsName)-$(TargetArchitecture)</TargetRuntimeIdentifier>
@@ -119,6 +120,7 @@
       win-arm;
       win-arm64;
       osx-x64;
+      freebsd-x64;
       linux-musl-x64;
       linux-musl-arm64;
       linux-x64;
-- 
2.21.0

