From 258c320eaaa7fec1651d10ceafa1b07e1762d05e Mon Sep 17 00:00:00 2001
From: Tomas Weinfurt <tweinfurt@yahoo.com>
Date: Wed, 13 Nov 2019 22:32:02 -0800
Subject: [PATCH 11/15] Add FreeBSD

---
 Directory.Build.props | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index be7a5a3..5ab07ca 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -115,6 +115,7 @@
   <PropertyGroup>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('Windows'))">win</TargetOsName>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('OSX'))">osx</TargetOsName>
+    <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('FreeBSD'))">freebsd</TargetOsName>
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('Linux'))">linux</TargetOsName>
     <BuildArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</BuildArchitecture>
     <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">x64</TargetArchitecture>
@@ -127,6 +128,7 @@
       win-arm;
       win-arm64;
       osx-x64;
+      freebsd-x64;
       linux-musl-x64;
       linux-musl-arm64;
       linux-x64;
-- 
2.21.0

