From 3c1a0b97b6bdd82508a657e96e1b62db3f9eb4d1 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 5 Jun 2019 21:26:29 -0500
Subject: [PATCH] Disable warn as error.

---
 Directory.Build.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 140f41d..8631d20 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -117,7 +117,7 @@
   <!-- Set up handling of build warnings -->
   <PropertyGroup>
     <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
   </PropertyGroup>
 
   <Import Project="$(RepositoryEngineeringDir)ReferenceAssemblies.props" />
-- 
2.18.0

