From 192ec5e9b6be4f806af0916269c0b49db71e2f67 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 9 Dec 2019 00:12:44 -0600
Subject: [PATCH 2/2] Package name was changed

---
 eng/Tools.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Tools.props b/eng/Tools.props
index 9425706..1f3f77e 100644
--- a/eng/Tools.props
+++ b/eng/Tools.props
@@ -19,7 +19,7 @@
 
     <!-- SourceLink -->
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
-    <PackageReference Include="Microsoft.SourceLink.Vsts.Git" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.SourceLink.AzureDevOpsServer.Git" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
   <!-- excluded from source build -->
-- 
2.18.0

