From ee53623c1b1fe37572a3e66b3466d2b4bcaf9601 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 24 Jun 2019 17:02:08 -0500
Subject: [PATCH] Add PVP import.

---
 eng/Versions.props | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 7e39b5e..21ce073 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -24,7 +24,7 @@
     <PreReleaseVersionLabel>preview9</PreReleaseVersionLabel>
   </PropertyGroup>
   <PropertyGroup>
-    <RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json;
@@ -48,4 +48,6 @@
     <MicrosoftAspNetCoreMvcAnalyzersPackageVersion>3.0.0-preview9.19413.1</MicrosoftAspNetCoreMvcAnalyzersPackageVersion>
     <MicrosoftAspNetCoreMvcApiAnalyzersPackageVersion>3.0.0-preview9.19413.1</MicrosoftAspNetCoreMvcApiAnalyzersPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
 </Project>
-- 
2.18.0

