From eabc084a191c8bb7a73c8f10a0d6b34b35b89647 Mon Sep 17 00:00:00 2001
From: Dan Seefeldt <dseefeld@microsoft.com>
Date: Wed, 7 Mar 2018 16:53:00 +0000
Subject: [PATCH] Pick up toolset from fixed location

---
 build.sh | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/build.sh b/build.sh
index d664515..f8ceca5 100755
--- a/build.sh
+++ b/build.sh
@@ -23,20 +23,13 @@ export WebSdkBuild=$WebSdkRoot/build/
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
+cp -r $DOTNET_TOOL_DIR/* $DOTNET_INSTALL_DIR
+
 [ -z $NUGET_PACKAGES ] && export NUGET_PACKAGES="$REPOROOT/.nuget/packages"
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.sh"
-curl -sSL "$DOTNET_INSTALL_SCRIPT_URL" | bash /dev/stdin --verbose --version 1.1.0
-
-curl --retry 10 -s -SL -f --create-dirs -o $DOTNET_INSTALL_DIR/buildtools.tar.gz https://aspnetcore.blob.core.windows.net/buildtools/netfx/4.6.1/netfx.4.6.1.tar.gz
-[ -d "$DOTNET_INSTALL_DIR/buildtools/net461" ] || mkdir -p $DOTNET_INSTALL_DIR/buildtools/net461
-tar -zxf $DOTNET_INSTALL_DIR/buildtools.tar.gz -C $DOTNET_INSTALL_DIR/buildtools/net461
-
 # Put stage 0 on the PATH (for this shell only)
 PATH="$DOTNET_INSTALL_DIR:$PATH"
-export ReferenceAssemblyRoot=$DOTNET_INSTALL_DIR/buildtools/net461
-
 
 # Increases the file descriptors limit for this bash. It prevents an issue we were hitting during restore
 FILE_DESCRIPTOR_LIMIT=$( ulimit -n )
@@ -46,4 +39,4 @@ then
     ulimit -n 1024
 fi
 
-$DOTNET_INSTALL_DIR/dotnet msbuild "$REPOROOT/build/build.proj" /t:Build /p:Configuration=$BuildConfiguration /p:NETFrameworkSupported=false $*
\ No newline at end of file
+$DOTNET_INSTALL_DIR/dotnet msbuild "$REPOROOT/build/build.proj" /t:Build /p:Configuration=$BuildConfiguration /p:NETFrameworkSupported=false $*
-- 
1.8.3.1

