From ee998e3d1261937bdad2a1fc294110c65ed60367 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Wed, 28 Aug 2019 14:26:39 -0500
Subject: [PATCH] Add PVP import.

---
 eng/Versions.props | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 51bfc96..cdd49dd 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -24,7 +24,7 @@
     <PreReleaseVersionLabel>preview9</PreReleaseVersionLabel>
   </PropertyGroup>
   <PropertyGroup>
-    <RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json;
@@ -48,4 +48,6 @@
     <MicrosoftAspNetCoreMvcAnalyzersPackageVersion>3.0.0-preview9.19424.4</MicrosoftAspNetCoreMvcAnalyzersPackageVersion>
     <MicrosoftAspNetCoreMvcApiAnalyzersPackageVersion>3.0.0-preview9.19424.4</MicrosoftAspNetCoreMvcApiAnalyzersPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
 </Project>
-- 
2.18.0

