From c9554f1c17f72ef5e17b9d1d932114e7b67fef42 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 24 Jun 2019 17:02:08 -0500
Subject: [PATCH 2/2] Add PVP import.

---
 eng/Versions.props | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index f645c42..a7b3b08 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -18,12 +18,15 @@
     <SystemCollectionsSpecializedPackageVersion>4.3.0</SystemCollectionsSpecializedPackageVersion>
     <SystemXmlXmlDocumentPackageVersion>4.3.0</SystemXmlXmlDocumentPackageVersion>
   </PropertyGroup>
+
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="Exists('$(DotNetPackageVersionPropsPath)')" />
+
   <PropertyGroup>
     <VersionPrefix>3.0.100</VersionPrefix>
     <PreReleaseVersionLabel>preview6</PreReleaseVersionLabel>
   </PropertyGroup>
   <PropertyGroup>
-    <RestoreSources>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json;
-- 
2.18.0

