From 9f2e482f30a47f4fc1845b90c63e511679173a31 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 3 Apr 2019 19:49:30 +0000
Subject: [PATCH] Remove PackageLicenseUrl

---
 eng/Package.props | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/eng/Package.props b/eng/Package.props
index 0308eb5..da7c8ec 100644
--- a/eng/Package.props
+++ b/eng/Package.props
@@ -10,8 +10,7 @@
   <PropertyGroup>
     <Description>This package contains MSBuild targets and tasks for building and publishing ASP.NET Core projects. This package is required for building and publishing ASP.NET Core projects.</Description>
     <ProjectUrl>https://github.com/aspnet/websdk</ProjectUrl>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?LinkId=320539</PackageLicenseUrl>
     <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
     <PackageTags>WebSdk</PackageTags>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
1.8.3.1

