From 57b65493ae7d4be6e7a88318e98877127b60a30c Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Mon, 28 Jan 2019 13:20:12 -0600
Subject: [PATCH] Add PackageProjectUrl to fix new SDK error.

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 196118d..39d91c8 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -14,6 +14,7 @@
 
   <PropertyGroup Label="Package and Assembly Metadata">
     <Product>Microsoft ASP.NET Core</Product>
+    <PackageProjectUrl>https://github.com/aspnet/websdk</PackageProjectUrl>
   </PropertyGroup>
 
   <PropertyGroup Label="Warning Suppressions">
-- 
2.18.0

