From 7d9eea7036e3de1d3c60792c16ae1e9150da395b Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 26 Apr 2019 00:15:49 -0500
Subject: [PATCH] Remove deprecated PackageLicenseUrl to fix warning.

---
 eng/Package.props | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/eng/Package.props b/eng/Package.props
index 0308eb5..da7c8ec 100644
--- a/eng/Package.props
+++ b/eng/Package.props
@@ -10,8 +10,7 @@
   <PropertyGroup>
     <Description>This package contains MSBuild targets and tasks for building and publishing ASP.NET Core projects. This package is required for building and publishing ASP.NET Core projects.</Description>
     <ProjectUrl>https://github.com/aspnet/websdk</ProjectUrl>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?LinkId=320539</PackageLicenseUrl>
     <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
     <PackageTags>WebSdk</PackageTags>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
2.18.0

