From 4c54fc0647963abe33c28280b52736a3054efcac Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Mon, 10 Jul 2017 19:11:02 -0700
Subject: [PATCH] Build with dotnet

---
 .../Microsoft.DiaSymReader.PortablePdb.csproj      | 82 ++++------------------
 1 file changed, 14 insertions(+), 68 deletions(-)

diff --git a/src/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.csproj b/src/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.csproj
index 61be78f..2954c63 100644
--- a/src/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.csproj
+++ b/src/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.csproj
@@ -1,73 +1,19 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup>
-    <Import Project="..\..\build\Targets\Settings.targets" />
-  </ImportGroup>
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectGuid>{F83343BA-B4EA-451C-B6DB-5D645E6171BC}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>Microsoft.DiaSymReader.PortablePdb</RootNamespace>
-    <AssemblyName>Microsoft.DiaSymReader.PortablePdb</AssemblyName>
+    <Version>1.2.0</Version>
+    <FileVersion>1.2.0.61320</FileVersion>
+    <InformationalVersion>$(FileVersion)</InformationalVersion>
+    <TargetFramework>netstandard1.1</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
-    <TargetFrameworkProfile>
-    </TargetFrameworkProfile>
-    <TargetFrameworkIdentifier>.NETPortable</TargetFrameworkIdentifier>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.1</NuGetTargetMoniker>
-    <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)../../build/Signing/35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
+    <PublicSign>true</PublicSign>
+    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
+
   <ItemGroup>
-    <Compile Include="AsyncMethodData.cs" />
-    <Compile Include="InvalidInputDataException.cs" />
-    <Compile Include="MethodExtent.cs" />
-    <Compile Include="MethodExtents.cs" />
-    <Compile Include="MethodLineDeltas.cs" />
-    <Compile Include="MethodMap.cs" />
-    <Compile Include="Utilities\BlobWriter.cs" />
-    <Compile Include="MethodId.cs" />
-    <Compile Include="DocumentId.cs" />
-    <Compile Include="Utilities\EmptyArray.cs" />
-    <Compile Include="Utilities\ExceptionUtilities.cs" />
-    <Compile Include="Utilities\HashHelpers.cs" />
-    <Compile Include="Utilities\ImmutableByteArrayInterop.cs" />
-    <Compile Include="Utilities\ImmutableArrayExtensions.cs" />
-    <Compile Include="Utilities\LazyMetadataImport.cs" />
-    <Compile Include="Utilities\FileNameUtilities.cs" />
-    <Compile Include="Utilities\HResult.cs" />
-    <Compile Include="Utilities\InteropUtilities.cs" />
-    <Compile Include="Utilities\KeyValuePair.cs" />
-    <Compile Include="Utilities\MetadataUtilities.cs" />
-    <Compile Include="Utilities\EnumerableHelpers.cs" />
-    <Compile Include="DocumentMap.cs" />
-    <Compile Include="IMetadataImport.cs" />
-    <Compile Include="PortablePdbReader.cs" />
-    <Compile Include="ScopeData.cs" />
-    <Compile Include="RootScopeData.cs" />
-    <Compile Include="ChildScopeData.cs" />
-    <Compile Include="SymBinder.cs" />
-    <Compile Include="SymScope.cs" />
-    <Compile Include="SymConstant.cs" />
-    <Compile Include="SymDocument.cs" />
-    <Compile Include="SymMethod.cs" />
-    <Compile Include="SymReader.cs" />
-    <Compile Include="SymVariable.cs" />
-    <Compile Include="Utilities\PortableShim.cs" />
-    <Compile Include="Utilities\ReadOnlyInteropStream.cs" />
-    <Compile Include="Utilities\ReflectionUtilities.cs" />
-    <Compile Include="Utilities\StreamExtensions.cs" />
-    <Compile Include="Utilities\TupleElementNamesAttribute.cs" />
-    <Compile Include="Utilities\ValueTuple.cs" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.4.2" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
+    <PackageReference Include="Microsoft.DiaSymReader" Version="1.1.0" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
-  <ItemGroup>
-    <InternalsVisibleTo Include="Microsoft.DiaSymReader.PortablePdb.UnitTests" />
-  </ItemGroup>
-  <ImportGroup Label="Targets">
-    <Import Project="..\..\build\Targets\Imports.targets" />
-  </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
1.8.3.1

